\documentclass[11pt, a4paper]{article}
\usepackage[a4paper, margin=1in]{geometry}

\usepackage{fontspec} 
\setmainfont{Roboto}

\usepackage[style=nature]{biblatex}
\addbibresource{prospectus.bib}

\usepackage{enumitem}

\usepackage{graphicx}

\usepackage{hyperref}

\begin{document}

\begin{titlepage}
\begin{tabular}{ r l }
 proposed title:   & I do some digital plumbing \\
 		   & \\
 author:	   & James Chuang \\
 		   & \\
 research advisor: & Fred Winston \\
 		   & \\
 abstract:	   & Lorem ipsum.
\end{tabular}
\end{titlepage}

\tableofcontents

\section{genomic approaches to studying the transcription elongation factor Spt6}

N.B. The work described in this section is currently in the review process. A preprint can be found at \url{https://www.biorxiv.org/content/early/2018/06/15/347575)} \cite{doris2018}.

\subsection{introduction}

The transcriptional landscape of the eukaryotic genome is complex, consisting of many classes of coding and non-coding transcripts which are often interleaved with one another \cite{jensen2013, pelechano2017}. All of these transcripts are regulated at the stage of transcription initiation, meaning that how a cell defines and regulates sites of initiation is fundamental to gene expression.

Past genetic studies in yeast produced the unexpected finding that the specificity of transcription initiation may be controlled in part by transcription \textit{elongation} factors, including histone chaperones and histone modification enzymes \cite{kaplan2003, cheung2008, hennig2013}. Therefore, transcription and co-transcriptional processes influence which sites in the genome are permitted to become sites of transcription initiation. 

One of the transcription elongation factors most important for the specificity of initiation sites is \textbf{Spt6}, which:
\begin{itemize}[nosep]
\item interacts directly with:
	\begin{itemize}[nosep]
	\item RNA polymerase II (RNAPII) \cite{close2011, diebold2011, liu2011, sdano2017, sun2010, yoh2007}
	\item histones \cite{bortvin1996, mccullough2015}
	\item the essential factor Spn1 (IWS1) \cite{diebold2010b, li2018, mcdonald2010}
	\end{itemize}
\item is believed to function primarily as an elongation factor based on:
	\begin{itemize}[nosep]
	\item association with elongating RNAPII \cite{andrulis2000, ivanovska2011, kaplan2000, mayer2010}
	\item ability to enhance elongation in vitro \cite{endoh2004} and in vivo \cite{ardehali2009}
	\end{itemize}
\item has been shown to regulate initiation in some cases \cite{adkins2006, ivanovska2011}
\item regulates:
	\begin{itemize}[nosep]
	\item chromatin structure \cite{bortvin1996, degennaro2013, ivanovska2011, jeronimo2015, kaplan2003, perales2013, vanbakel2013}
	\item histone modifications, including:
		\begin{itemize}[nosep]
		\item H3K36 methylation \cite{carrozza2005, chu2006, yoh2008, youdell2008}
		\item in some organisms, H3K4 and H3K27 methylation \cite{begum2012, chen2012, degennaro2013, wang2017, wang2013}
		\end{itemize}
	\end{itemize}
\item is likely to be a histone chaperone required to reassemble nucleosomes in the wake of transcription \cite{duina2011}
\end{itemize}

Studies in the yeasts \textit{S. cerevisiae} and \textit{S. pombe} have shown that Spt6 controls transcription genome-wide \cite{cheung2008, degennaro2013, kaplan2003, pathak2018, uwimana2017, vanbakel2013}. In \textit{spt6} mutants, transcription changes dramatically, with altered sense transcription and increased levels of transcription antisense to genes. Notably, in \textit{spt6} mutants there is widespread upregulation of cryptic or \textbf{intragenic transcription} appearing to initiate from within protein-coding genes \cite{cheung2008, degennaro2013, kaplan2003, uwimana2017}.

\begin{figure}[h]
\centering
\includegraphics[width=0.8\textwidth]{figures/spt6_2018_figure1C-TSS-seq-diffexp-summary.pdf}
\end{figure}

In this project, we address issues relating to intragenic transcription and its regulation by Spt6 in \textit{Saccharomyces cerevisiae}. Previous studies of transcript levels in \textit{S. cerevisiae spt6} mutants used Northern blots \cite{kaplan2003}, tiled microarrays \cite{cheung2008}, and RNA-sequencing \cite{uwimana2017}. Because these techniques measure steady-state RNA levels, they are unable to distinguish whether intragenic transcripts are the result of new initation, or the result of changes in RNA processing or decay. Microarrays and RNA-seq are also unable to detect intragenic transcripts from highly transcribed genes, because the signal from any intragenic transcript is convoluted with the signal from the overlapping `genic' transcript \cite{cheung2008, lickwar2009}.

\subsection{results}
\subsection{discussion}

\section{searching for functions of intragenic transcription in stress}
\subsection{introduction}
\subsection{results}
\subsection{discussion}

Lorem ipsum.

\printbibliography
\end{document}
