\documentclass[11pt, a4paper]{article}
\usepackage[a4paper, margin=1in]{geometry}

\usepackage{fontspec}
\setmainfont{Roboto-Regular}[
    Path=fonts/,
    BoldFont=Roboto-Bold,
    ItalicFont=Roboto-Italic,
    BoldItalicFont=Roboto-BoldItalic
]

% \usepackage[style=nature]{biblatex}
\usepackage[]{cite}
% \addbibresource{prospectus.bib}

\usepackage{enumitem}

\usepackage{graphicx}

\usepackage{hyperref}

\usepackage{textcomp}

\begin{document}

\begin{titlepage}
\begin{tabular}{ r l }
 proposed title:   & I do some digital plumbing \\
 		   & \\
 author:	   & James Chuang \\
 		   & \\
 research advisor: & Fred Winston \\
 		   & \\
 abstract:	   & Lorem ipsum.
\end{tabular}
\end{titlepage}

\tableofcontents

\section{analysis of genomics data relating to the transcription elongation factor Spt6}

N.B. The work described in this section is currently in the review process. A preprint can be found at \url{https://doi.org/10.1101/347575} \cite{doris2018}.

\subsection{introduction to Spt6, intragenic transcription, and assays used to study the two}

The aim of the work described in this section is related to understanding how a eukaryotic cell specifies which sites in its genome are permitted to become sites of transcription initiation. From past genetic studies in yeast, it is known that some of the factors involved in controlling the specificity of transcription initiation are actually transcription \textit{elongation} factors, including histone chaperones and histone modification enzymes \cite{kaplan2003, cheung2008, hennig2013}. My collaborators on this project are interested in the role of the transcription elongation factor \textbf{Spt6} in this process.

\begin{itemize}[nosep, topsep=.5em]
\item Spt6 interacts directly with:
	\begin{itemize}[nosep]
	\item RNA polymerase II (RNAPII) \cite{close2011, diebold2011, liu2011, sdano2017, sun2010, yoh2007}
	\item histones \cite{bortvin1996, mccullough2015}
	\item the essential factor Spn1 (IWS1) \cite{diebold2010b, li2018, mcdonald2010}
	\end{itemize}
\item Spt6 is believed to function primarily as an elongation factor based on:
	\begin{itemize}[nosep]
	\item association with elongating RNAPII \cite{andrulis2000, ivanovska2011, kaplan2000, mayer2010}
	\item ability to enhance elongation in vitro \cite{endoh2004} and in vivo \cite{ardehali2009}
	\end{itemize}
\item Spt6 has been shown to regulate initiation in some cases \cite{adkins2006, ivanovska2011}
\item Spt6 regulates:
	\begin{itemize}[nosep]
	\item chromatin structure \cite{bortvin1996, degennaro2013, ivanovska2011, jeronimo2015, kaplan2003, perales2013, vanbakel2013}
	\item histone modifications, including:
		\begin{itemize}[nosep]
		\item H3K36 methylation \cite{carrozza2005, chu2006, yoh2008, youdell2008}
		\item in some organisms, H3K4 and H3K27 methylation \cite{begum2012, chen2012, degennaro2013, wang2017, wang2013}
		\end{itemize}
	\end{itemize}
\item Spt6 is likely to be a histone chaperone required to reassemble nucleosomes in the wake of transcription \cite{duina2011}.
\end{itemize}

Previous studies in the yeasts \textit{S. cerevisiae} and \textit{S. pombe} have examined the requirement for Spt6 in normal transcription \cite{cheung2008, degennaro2013, kaplan2003, pathak2018, uwimana2017, vanbakel2013}. Many of these studies make use of the same temperature-sensitive \textit{S. cerevisiae} \textit{spt6} mutant used in this project, \textbf{\textit{spt6-1004}}, in which Spt6 protein is depleted at the non-permissive temperature of 37\textdegree C. The most notable phenotype of the \textit{spt6-1004} mutant is the appearance of \textbf{intragenic transcripts}, transcripts which appear to initiate from within protein-coding genes, both in same orientation and in the antisense orientation relative to the coding gene \cite{cheung2008, degennaro2013, kaplan2003, uwimana2017}.

Previous genome-wide measurements of transcript levels in \textit{spt6-1004} were made using tiled microarrays \cite{cheung2008} and RNA sequencing \cite{uwimana2017}. These methods measure steady-state RNA levels, making them unable to determine whether the intragenic transcripts observed in \textit{spt6-1004} are the result of: A) new intragenic transcription initiation in the mutant, B) reduced decay of an intragenic transcript normally turned over rapidly in wild-type, or C) RNA processing of the full-length protein-coding RNA. Additionally, these methods are suboptimal for identifying where intragenic transcription occurs, since the signal for an intragenic transcript in the same orientation as the gene it overlaps is convoluted with the signal from the full-length `genic' transcript \cite{cheung2008, lickwar2009}.

To overcome these issues, one of my collaborators applied two genomic assays to study transcription in \textit{spt6-1004}: transcription start-site sequencing (TSS-seq), and ChIP-nexus of TFIIB, a component of the RNA polymerase II (RNAPII) pre-initiation complex (PIC). The TSS-seq technique used sequences the 5' end of capped and polyadenylated RNAs, allowing us to separate intragenic from genic RNA signals and identify the locations of intragenic transcripts with single-nucleotide resolution. The ChIP-nexus technique used is a high-resolution chromatin immunoprecipitation technique in which the ChIPed DNA is exonuclease digested up to the bases crosslinked with the factor of interest before sequencing. When applied to the PIC component TFIIB, ChIP-nexus allows us to test if intragenic transcripts actually result from new intragenic transcription initiation.

\subsection{pipeline development for TSS-seq and ChIP-nexus}

In order to analyze TSS-seq and ChIP-nexus data and answer questions about Spt6 and intragenic transcription, I developed analysis pipelines for TSS-seq and ChIP-nexus data. The pipelines are written using the Python-based Snakemake workflow specification language, and carry out steps including read cleaning, various quality controls, read alignment, data normalization, coverage track generation, peak calling, differential expression/binding analyses, data visualization, motif enrichment analyses, and gene ontology analyses. Importantly, the Snakemake framework allows data analyses using these pipelines to be reproducible and scalable from workstations up to computing clusters. Up-to-date versions of these pipelines with more details on their capabilities are available at \href{https://github.com/winston-lab}{github.com/winston-lab}. In the following subsections I will only describe the thought behind a few of the more novel pipeline steps before moving on to results relating to Spt6 and intragenic transcription.

\subsubsection{TSS-seq peak calling}

\subsubsection{ChIP-nexus peak calling}

% \begin{figure}[h]
% \centering
% \includegraphics[width=0.8\textwidth]{figures/spt6_2018_figure1C-TSS-seq-diffexp-summary.pdf}
% \end{figure}


\subsection{results}
\subsection{discussion}

\section{searching for functions of intragenic transcription in stress}
\subsection{introduction}
\subsection{results}
\subsection{discussion}

\section{Spt5 does things}

Lorem ipsum.

% \printbibliography
\bibliography{prospectus}{}
\bibliographystyle{plain}
\end{document}
