\documentclass[11pt, a4paper]{article}
\usepackage[a4paper, margin=18mm]{geometry}

\usepackage{fontspec}
\setmainfont{Roboto-Regular}[
    Path=fonts/,
    BoldFont=Roboto-Bold,
    ItalicFont=Roboto-Italic,
    BoldItalicFont=Roboto-BoldItalic
]

% \usepackage[style=nature]{biblatex}
\usepackage[]{natbib}
% \addbibresource{prospectus.bib}

\usepackage{enumitem}
\usepackage{graphicx}
\usepackage{hyperref}
\usepackage{textcomp}
\usepackage{caption}
\captionsetup{font=footnotesize}
\usepackage{lipsum}
% \usepackage{wrapfig}

\begin{document}

\begin{titlepage}
\begin{tabular}{ r l }
 proposed title:   & Digital plumbing in the genomics era. \\
 		   & \\
 author:	   & James Chuang \\
 		   & \\
 research advisor: & Fred Winston \\
 		   & \\
 abstract:	   & Lorem ipsum.
\end{tabular}
\end{titlepage}

\tableofcontents
\newpage

\section{analysis of genomics data relating to the transcription elongation factor Spt6}

N.B. The work described in this section is currently in review. A preprint can be found at \url{https://doi.org/10.1101/347575} \cite{doris2018}.

\subsection{introduction to Spt6, intragenic transcription, and assays used to study the two}

The aim of the work described in this section is related to understanding how a eukaryotic cell specifies which sites in its genome are permitted to become sites of transcription initiation. From past genetic studies in yeast, it is known that some of the factors involved in controlling the specificity of transcription initiation are actually transcription \textit{elongation} factors, including histone chaperones and histone modification enzymes \cite{kaplan2003, cheung2008, hennig2013}. My collaborators on this project are interested in the role of the transcription elongation factor \textbf{Spt6} in this process. The following is a quick introduction to Spt6:

\begin{itemize}[nosep, topsep=.5em]
\item Spt6 interacts directly with:
	\begin{itemize}[nosep]
	\item RNA polymerase II (RNAPII) \cite{close2011, diebold2011, liu2011, sdano2017, sun2010, yoh2007}
	\item histones \cite{bortvin1996, mccullough2015}
	\item the essential factor Spn1 (IWS1) \cite{diebold2010b, li2018, mcdonald2010}
	\end{itemize}
\item Spt6 is believed to function primarily as an elongation factor based on:
	\begin{itemize}[nosep]
	\item association with elongating RNAPII \cite{andrulis2000, ivanovska2011, kaplan2000, mayer2010}
	\item ability to enhance elongation in vitro \cite{endoh2004} and in vivo \cite{ardehali2009}
	\end{itemize}
\item Spt6 has been shown to regulate initiation in some cases \cite{adkins2006, ivanovska2011}
\item Spt6 regulates:
	\begin{itemize}[nosep]
	\item chromatin structure \cite{bortvin1996, degennaro2013, ivanovska2011, jeronimo2015, kaplan2003, perales2013, vanbakel2013}
	\item histone modifications, including:
		\begin{itemize}[nosep]
		\item H3K36 methylation \cite{carrozza2005, chu2006, yoh2008, youdell2008}
		\item in some organisms, H3K4 and H3K27 methylation \cite{begum2012, chen2012, degennaro2013, wang2017, wang2013}
		\end{itemize}
	\end{itemize}
\item Spt6 is likely to be a histone chaperone required to reassemble nucleosomes in the wake of transcription \cite{duina2011}.
\end{itemize}

Previous studies in the yeasts \textit{S. cerevisiae} and \textit{S. pombe} have examined the requirement for Spt6 in normal transcription \cite{cheung2008, degennaro2013, kaplan2003, pathak2018, uwimana2017, vanbakel2013}. Many of these studies make use of the same temperature-sensitive \textit{S. cerevisiae} \textit{spt6} mutant used in this project, \textbf{\textit{spt6-1004}}, in which Spt6 protein is depleted at the non-permissive temperature of 37\textdegree C. The most notable phenotype of the \textit{spt6-1004} mutant is the appearance of \textbf{intragenic transcripts}, transcripts which appear to start within protein-coding genes, both in the same orientation and in the antisense orientation relative to the coding gene \cite{cheung2008, degennaro2013, kaplan2003, uwimana2017}.

Previous genome-wide measurements of transcript levels in \textit{spt6-1004} were made using tiled microarrays \cite{cheung2008} and RNA sequencing \cite{uwimana2017}. These methods measure steady-state RNA levels, making them unable to determine whether the intragenic transcripts observed in \textit{spt6-1004} are the result of: A) new intragenic transcription initiation in the mutant, B) reduced decay of an intragenic transcript normally turned over rapidly in wild-type, or C) RNA processing of the full-length protein-coding RNA. Additionally, these methods are suboptimal for identifying where intragenic transcription occurs, since the signal for an intragenic transcript in the same orientation as the gene it overlaps is convoluted with the signal from the full-length `genic' transcript \cite{cheung2008, lickwar2009}.

To overcome these issues, one of my collaborators applied two genomic assays to study transcription in \textit{spt6-1004}: transcription start-site sequencing (TSS-seq), and ChIP-nexus of TFIIB, a component of the RNA polymerase II (RNAPII) pre-initiation complex (PIC). The TSS-seq technique used sequences the 5' end of capped and polyadenylated RNAs, allowing us to separate intragenic from genic RNA signals and identify the locations of intragenic transcripts with single-nucleotide resolution. The ChIP-nexus technique used is a high-resolution chromatin immunoprecipitation technique in which the ChIPed DNA is exonuclease digested up to the bases crosslinked with the factor of interest before sequencing. When applied to the PIC component TFIIB, ChIP-nexus allows us to test if intragenic transcripts actually result from new intragenic transcription initiation.

\subsection{pipeline development for TSS-seq and ChIP-nexus}

In order to analyze TSS-seq and ChIP-nexus data and answer questions about Spt6 and intragenic transcription, I developed analysis pipelines for TSS-seq and ChIP-nexus data. The pipelines are written using the Python-based Snakemake workflow specification language, and carry out steps including read cleaning, various quality controls, read alignment, data normalization, coverage track generation, peak calling, differential expression/binding analyses, data visualization, motif enrichment analyses, and gene ontology analyses. The Snakemake framework allows data analyses using these pipelines to be reproducible and scalable from workstations up to computing clusters. Up-to-date versions of these pipelines with more details on their capabilities are available at \href{https://github.com/winston-lab}{github.com/winston-lab}. In the following subsections I will only describe the thought behind a few of the more novel pipeline steps before moving on to results relating to Spt6 and intragenic transcription.

\subsubsection{TSS-seq peak calling}

TSS-seq data associated with a single region of transcription initiation tends to occur as a group of signal distributed over a range of positions, rather than signal at a single nucleotide (Figure \ref{fig:tss_coverage}). It is reasonable to consider such a group of TSS-seq signal as one entity, because the signals within the group are highly correlated to one another across different conditions. Therefore, to identify TSSs and quantify them for downstream analyses such as differential expression, it is necessary to perform peak-calling on TSS-seq data.

\begin{figure}[h]
\centering
\includegraphics[width=8cm]{figures/figure1_tss-seq-coverage.pdf}
\caption{Wild-type sense strand TSS-seq signal at the TEF1 genic TSS. Normalized counts are the mean of spike-in normalized coverage from two replicates.}
\label{fig:tss_coverage}
\end{figure}

The TSS-seq pipeline used currently performs peak calling using a 1-D watershed segmentation algorithm, followed by filtering for reproducibility by the Irreproducible Discovery Rate (IDR) method. First, a smoothed version of the TSS-seq coverage is generated for each sample using a discretized Gaussian kernel. Next, an initial set of peaks is generated by assigning all nonzero signal in the original, unsmoothed coverage to the nearest local maximum of the smoothed coverage, and taking the minimum and maximum genomic coordinates of the original coverage as the peak boundaries for each local maximum of the smoothed coverage. Peaks are then trimmed to the smallest genomic window that includes 95\% of the original coverage, and the probability of the peak being generated by noise is estimated by a Poisson model where $\lambda$, the expected coverage, is the maximum of the expected coverage over the chromosome and the expected coverage in a window upstream of the peak (as for the ChIP-seq peak caller MACS2). Letting the expected coverage be influenced by the local read density is intended to reduce false positive peaks within gene bodies, especially for highly expressed genes: Since there are more fragments of RNA present for highly expressed genes, more fragments within the gene body will make it into the final library, even if they are not true 5' ends. The final step in the current peak calling algorithm is to rank the initial set of peaks by their significance under the Poisson model, and filter this list of peaks using the IDR method. In brief, IDR works using the following intuition: True positive peaks, when ranked by some metric, will have similar ranks in two replicates, while peaks arising from experimental noise will have less similar ranks in two replicates.

Currently, the IDR algorithm only works for two replicates. Future improvements could include expanding the IDR implementation to take more than two replicates into account, which would improve the accuracy of peak calling given more data.

\subsubsection{ChIP-nexus peak calling}

\begin{figure}[h]
\centering
\includegraphics[width=7cm]{figures/figure3_tfiib-nexus-tata.pdf}
\caption{Average TFIIB ChIP-nexus signal from wild-type cells grown at 37\textdegree C for 80 minutes, aligned to 572 TATA boxes with no mismatches to the sequence TATAWAWR as previously defined by \citep{rhee2012}. The signal around each TATA box is scaled from 0 to 1 before taking the mean in order to normalize differences in levels of TFIIB binding. Crosslinking signal on the plus and minus strands are plotted above and below the x-axis, respectively.}
\label{fig:tfiib_tata}
\end{figure}

A number of tools have been created specifically for peak calling using high-resolution ChIP techniques such as ChIP-nexus and ChIP-exo. When applied to our TFIIB ChIP-nexus data, these tools tended to split what appeared to be a single TFIIB binding event into multiple peaks. This may be due to the fact that TFIIB has been observed to have multiple crosslinking points to the DNA \ref{fig:tfiib_tata} \cite{rhee2012}, and suggests that while these tools may work well for factors which bind symmetrically with a single crosslinking point on either side, there is still room for improvement when it comes to factors with more complex binding patterns. For the purposes of this project, the standard ChIP-seq peak caller MACS2 was used.

ChIP-seq peaks do not have strand information, as DNA binding factors usually don't bind DNA in a strand-specific manner. This precluded us from separating intragenic TFIIB peaks into peaks associated with sense or antisense transcription. The distinctive shape of the aggregate TFIIB ChIP-nexus signal suggests that information about the strand of transcription may be present in the ChIP-nexus binding profile. Future work could include learning the direction of transcription from labeled ChIP-nexus training data.

\subsection{TSS-seq and TFIIB ChIP-nexus results in \textit{spt6-1004}}

\begin{figure}[h]
\centering
\includegraphics[width=17.4cm]{figures/figure2_tss-seq-heatmaps.pdf}
\caption{Heatmaps of sense and antisense TSS-seq signal from wild-type and \textit{spt6-1004} cells, over 3522 non-overlapping genes aligned by wild-type genic TSS and sorted by annotated transcript length. Data are shown for each gene up to 300 nucleotides 3' of the cleavage and polyadenylation site (CPS, indicated by the white dotted line). Values are the mean of spike-in normalized coverage in non-overlapping 20 nucleotide bins, averaged over two replicates. Values above the 93rd percentile are set to the 93rd percentile for visualization.}
\label{fig:tss_heatmaps}
\end{figure}

\begin{figure}[h]
\centering
\includegraphics[width=8.7cm]{figures/figure4_tfiib-heatmaps.pdf}
\caption{Heatmaps of TFIIB binding measured by ChIP-nexus, over the same regions shown in Figure \ref{fig:tss_heatmaps}. Values are the mean of library-size normalized coverage in 20 basepair windows, averaged over two replicates. Values above the 85th percentile are set to the 85th percentile for visualization.}
\label{fig:tfiib_heatmaps}
\end{figure}

To assay transcription start sites and transcription initiation in \textit{spt6-1004}, one of my collaborators performed TSS-seq and ChIP-nexus of TFIIB. In wild-type cells, TSS-seq and TFIIB ChIP-nexus signal has the expected distribution over the genome, with most TSS-seq signal at annotated genic TSSs and most TFIIB signal just upstream (Figures \ref{fig:tss_heatmaps}, \ref{fig:tfiib_heatmaps}). In \textit{spt6-1004}, the signal for both assays infiltrates gene bodies, consistent with a role for intragenic initiation in the intragenic transcription phenotype. In particular, sense strand TSS-seq signal in \textit{spt6-1004} tends to occur towards the 3' end of genes, while antisense strand TSS-seq signal tends to occur towards the 5' ends of genes.

The TSS-seq data were quantified by peak calling and differential expression analysis, and classified into genomic categories based on their position relative to coding genes (Figure \ref{fig:tss_diffexp_summary}). The results from this analysis support the pattern observed in the heatmap visualization (Figure \ref{fig:tss_heatmaps}), with most genic TSSs downregulated and almost 8000 TSSs upregulated intragenic or antisense to genes. The overall effect of this on expression levels is to equalize expression levels between the different classes of transcripts (Figure \ref{fig:tss_expression_levels}).

\begin{figure}[h]
\centering
\includegraphics[width=8cm]{figures/figure5_tss-diffexp-summary.pdf}
\caption{Top) Diagram of different genomic classes of TSSs. Bottom) Bar plot showing the number of TSS-seq peaks differentially expressed in \textit{spt6-1004} versus wild-type.}
\label{fig:tss_diffexp_summary}
\end{figure}

\begin{figure}[h]
\centering
\includegraphics[width=7cm]{figures/figure6_tss-expression-levels.pdf}
\caption{Violin plots of expression level distributions for different genomic classes of TSS-seq peaks in wild-type and \textit{spt6-1004}. Normalized counts are the mean of \textit{S. pombe} spike-in size factor normalized counts from two replicates.}
\label{fig:tss_expression_levels}
\end{figure}

\begin{figure}[h]
\centering
\includegraphics[width=8cm]{figures/figure8_tfiib-spreading-ssa4.pdf}
\caption{Top) TFIIB ChIP-nexus protection in wild-type and \textit{spt6-1004} strains over 20 kb of chromosome II flanking the \textit{SSA4} gene. Bottom) Expanded view of TFIIB protection over the \textit{SSA4} gene.}
\end{figure}

\begin{figure}[h]
\centering
\includegraphics[width=17.4cm]{figures/figure7_tss-v-tfiib.pdf}
\caption{Scatterplots of fold-change in \textit{spt6-1004} over wild-type, comparing TSS-seq and TFIIB ChIP-nexus. Each dot represents a TSS-seq peak paired with the window extending 200 nucleotides upstream of the TSS-seq peak summit for quantification of TFIIB ChIP-nexus signal. Fold-changes are regularized fold-change estimates from DESeq2, with size factors determined from the \textit{S. pombe} spike-in (TSS-seq), or \textit{S. cerevisiae} counts (ChIP-nexus).}
\end{figure}

\lipsum


\section{searching for functions of intragenic transcription in stress}
\subsection{introduction}
\subsection{results}
\subsection{discussion}

\section{Spt5 does things}

\section{acknowledgements}

Steve Doris adapted the TSS-seq and ChIP-nexus techniques and produced the libraries for this work. Olga Viktorovskaya produced the MNase-seq libraries for this work. Magdalena Murawska produced the NET-seq libraries for this work.

% \printbibliography
\bibliography{prospectus}{}
\bibliographystyle{plain}
\end{document}
