\documentclass[9pt, letterpaper]{article}
\usepackage[letterpaper, margin=21mm]{geometry}

\usepackage{mathspec}
\setmainfont{Roboto-Regular}[
    Path=fonts/,
    BoldFont=Roboto-Bold,
    ItalicFont=Roboto-Italic,
    BoldItalicFont=Roboto-BoldItalic
]
% \setmathsfont(Digits)[Path=fonts/]{Roboto-Regular}
\setmathrm[Path=fonts/]{Roboto-Regular}

% \usepackage[style=nature]{biblatex}
\usepackage[]{natbib}
% \addbibresource{prospectus.bib}

\usepackage{morefloats}
\usepackage{enumitem}
\usepackage{graphicx}
\usepackage{float}
\usepackage{sidecap}
\usepackage{wrapfig}
\usepackage{hyperref}
\usepackage{textcomp}
\usepackage{caption}
\captionsetup{font=footnotesize, singlelinecheck=off}
\usepackage{lipsum}
% \usepackage{wrapfig}

\setlength\intextsep{4pt}

\begin{document}

\begin{titlepage}
\begin{tabular}{ r l }
 proposed title:   & Digital plumbing in the genomics era. \\
 		   & \\
 author:	   & James Chuang \\
 		   & \\
 research advisor: & Fred Winston \\
 		   & \\
 abstract:	   & Lorem ipsum.
\end{tabular}
\end{titlepage}

\tableofcontents
\newpage

\section{introduction}

In the past two years, I have taken on the three projects described in this document. My role in these projects is a mix of \href{https://blog.insightdatascience.com/data-science-vs-data-engineering-62da7678adaa}{\textbf{data scientist}} and \href{https://blog.insightdatascience.com/data-science-vs-data-engineering-62da7678adaa}{\textbf{data engineer}}: I build pipelines for processing large (in this case, genomic) datasets, from raw data to statistical analysis and data visualization. This mostly involves surveying available tools, selecting the tools most suitable for each step of a pipeline, and coding solutions to problems when existing tools are insufficient. All of my data analyses are open source (\href{https://github.com/winston-lab}{github.com/winston-lab}) and are designed to be reproducible by others: For every publication, I upload an archive that allows those interested to recreate the figures in the publication starting from raw data (e.g. \url{https://doi.org/10.5281/zenodo.1409826}).

\section{genomics of transcription elongation factor Spt6}

N.B. This work is in press. A preprint is available at \url{https://doi.org/10.1101/347575} \cite{doris2018}.

\subsection{collaborators}

\begin{description}[align=right, labelwidth=5cm, noitemsep]
    \item [Steve Doris] optimized TSS-seq and ChIP-nexus protocols
    \item [] generated TSS-seq and ChIP-nexus libraries
    \item [Olga Viktorovskaya] generated MNase-seq libraries
    \item [Magdalenda Murawska] generated NET-seq libraries
    \item [Dan Spatt] wetlab experiments for publication
\end{description}

\subsection{introduction to Spt6, intragenic transcription, and assays used to study the two}

The aim of this work is related to understanding how a eukaryotic cell specifies which sites in its genome are permitted to become sites of transcription initiation. From genetic studies in yeast, it is known that some factors controlling the specificity of transcription initiation are actually transcription \textit{elongation} factors, including histone chaperones and histone modification enzymes \cite{kaplan2003, cheung2008, hennig2013}. My collaborators on this project are interested in the role of the transcription elongation factor \textbf{Spt6} in this process. The following is a quick introduction to Spt6 \cite{doris2018}:

\begin{itemize}[nosep, topsep=.5em]
\item Spt6 interacts directly with:
	\begin{itemize}[nosep]
	\item RNA polymerase II (RNAPII) \cite{close2011, diebold2011, liu2011, sdano2017, sun2010, yoh2007}
	\item histones \cite{bortvin1996, mccullough2015}
	\item the essential factor Spn1 (IWS1) \cite{diebold2010b, li2018, mcdonald2010}
	\end{itemize}
\item Spt6 is believed to function primarily as an elongation factor based on:
	\begin{itemize}[nosep]
	\item association with elongating RNAPII \cite{andrulis2000, ivanovska2011, kaplan2000, mayer2010}
	\item ability to enhance elongation in vitro \cite{endoh2004} and in vivo \cite{ardehali2009}
	\end{itemize}
\item Spt6 has been shown to regulate initiation in some cases \cite{adkins2006, ivanovska2011}
\item Spt6 regulates:
	\begin{itemize}[nosep]
	\item chromatin structure \cite{bortvin1996, degennaro2013, ivanovska2011, jeronimo2015, kaplan2003, perales2013, vanbakel2013}
	\item histone modifications, including:
		\begin{itemize}[nosep]
		\item H3K36 methylation \cite{carrozza2005, chu2006, yoh2008, youdell2008}
		\item in some organisms, H3K4 and H3K27 methylation \cite{begum2012, chen2012, degennaro2013, wang2017, wang2013}
		\end{itemize}
	\end{itemize}
\item Spt6 is likely a histone chaperone required to reassemble nucleosomes in the wake of transcription \cite{duina2011}.
\end{itemize}

\begin{wrapfigure}[9]{r}{7.02cm}
\centering
\includegraphics[width=7cm]{figures/figure0_txn-diagram.pdf}
\caption{Diagram of transcript orientation with respect to coding DNA sequences, for the categories of transcripts referred to in this document.}
\label{fig:txn-diagram}
\end{wrapfigure}

Previous studies in the yeasts \textit{S. cerevisiae} and \textit{S. pombe} have examined the requirement for Spt6 in normal transcription \cite{cheung2008, degennaro2013, kaplan2003, pathak2018, uwimana2017, vanbakel2013}. Many of these studies make use of the same temperature-sensitive \textit{S. cerevisiae} \textit{spt6} mutant used in this project, \textbf{\textit{spt6-1004}}, in which Spt6 protein is depleted at the non-permissive temperature of 37\textdegree C \cite{kaplan2003}. The most notable phenotype of the \textit{spt6-1004} mutant is the expression of \textbf{intragenic transcripts}, transcripts which appear to start within protein-coding genes, both in the same orientation and in the antisense orientation relative to the coding gene (Figure \ref{fig:txn-diagram}) \cite{cheung2008, degennaro2013, kaplan2003, uwimana2017}.

Previous genome-wide measurements of transcript levels in \textit{spt6-1004} relied on tiled microarrays \cite{cheung2008} and RNA sequencing \cite{uwimana2017}. These methods assay steady-state RNA levels, making them unable to determine whether the intragenic transcripts observed in \textit{spt6-1004} result from: A) new intragenic transcription initiation in the mutant, B) reduced decay of intragenic transcripts which are rapidly turned over in wild-type, or C) processing of full-length protein-coding RNAs. Additionally, these methods are suboptimal for identifying where intragenic transcription occurs, since the signal for an intragenic transcript in the same orientation as the gene it overlaps is convoluted with the signal from the full-length `genic' transcript (Figure \ref{fig:txn-diagram}) \cite{cheung2008, lickwar2009}.

To overcome these issues, one of my collaborators applied two assays to study transcription in \textit{spt6-1004}: transcription start-site sequencing (\textbf{TSS-seq}), and \textbf{ChIP-nexus of TFIIB}, a component of the RNA polymerase II pre-initiation complex (PIC). The TSS-seq technique sequences the 5' end of capped and polyadenylated RNAs \cite{arribere2013, malabat2015}, allowing separation of intragenic from genic RNA signals and identification of intragenic transcript starts with single-nucleotide resolution. The ChIP-nexus technique used is a high-resolution chromatin immunoprecipitation technique in which the ChIPed DNA is exonuclease digested up to the bases crosslinked with the factor of interest before sequencing \cite{he2015}. When applied to the PIC component TFIIB, ChIP-nexus provides a way to determine whether intragenic transcripts result from new intragenic transcription initiation.

\subsection{pipeline development for TSS-seq and ChIP-nexus}

In order to use TSS-seq and ChIP-nexus to answer questions about Spt6 and intragenic transcription, I developed analysis pipelines for TSS-seq and ChIP-nexus data. The pipelines are written using the Python-based Snakemake workflow specification language \cite{koster2012}, and perform steps including read cleaning \cite{martin2011}, various quality controls \cite{andrews2012}, read alignment \cite{kim2013, langmead2012}, data normalization, coverage track generation \cite{quinlan2010}, peak calling \cite{zhang2008}, differential expression/binding analyses \cite{love2014}, data visualization with clustering, motif enrichment analyses \cite{bailey2015}, and gene ontology analyses \cite{young2010}. The Snakemake framework allows thesee data analyses to be reproducible and scalable from workstations up to computing clusters. Updated versions of these pipelines with more details on their capabilities are available at \href{https://github.com/winston-lab}{github.com/winston-lab}. In the following subsections I will describe the thought behind only a few of the more novel pipeline steps before moving on to results relating to Spt6 and intragenic transcription.

\subsubsection{TSS-seq peak calling}

TSS-seq data from a single region of transcription initiation tends to occur as a cluster of signal distributed over a range of positions, rather than a single nucleotide (Figure \ref{fig:tss_coverage}) \cite{arribere2013, malabat2015}. It is reasonable to consider such a cluster of TSS-seq signal as a single entity, because the signals within the cluster are usually highly correlated to one another across different conditions. Therefore, to identify TSSs from TSS-seq data and quantify them for downstream analyses such as differential expression, it is necessary to annotate these groups of signal by using the data to perform peak-calling.

\begin{wrapfigure}[12]{l}{7.02cm}
\centering
\includegraphics[width=7cm]{figures/figure1_tss-seq-coverage.pdf}
\caption{Wild-type sense strand TSS-seq signal at the \textit{TEF1} genic TSS. Normalized counts are the mean of spike-in normalized coverage from two replicates.}
\label{fig:tss_coverage}
\end{wrapfigure}

In its current state, the TSS-seq pipeline calls peaks using 1-D \href{https://en.wikipedia.org/wiki/Watershed_(image_processing)}{watershed segmentation}, followed by filtering for reproducibility by the Irreproducible Discovery Rate (IDR) method \cite{li2011}. First, a smoothed version of the TSS-seq coverage is generated for each sample using a discretized Gaussian kernel. Next, an initial set of peaks is generated by: 1) assigning all nonzero signal in the original, unsmoothed coverage to the nearest local maximum of the smoothed coverage in the direction of positive derivative, and 2) taking the minimum and maximum genomic coordinates of the original coverage assigned to each local maximum as the peak boundaries. The peaks are then trimmed to the smallest genomic window that includes 95\% of the original coverage, and the probability of the peak being generated by noise is estimated by a Poisson model where $\lambda$, the expected coverage, is the maximum of the expected coverage over the chromosome and the expected coverage in a window upstream of the peak (as for the ChIP-seq peak caller MACS2 \cite{zhang2008}). The influence of local read density on $\lambda$ is intended to reduce false positive peaks within gene bodies, especially for highly expressed genes: Since there are more fragments of RNA present for highly expressed genes, more fragments within the gene body will make it into the final library, even if they are not true 5' ends. To generate the final set of peaks, the peaks are ranked by significance under the Poisson model, and filtered by IDR. In brief, IDR attempts to separate true peaks from experimental noise based on the intuition that, when peaks in each replicate are independently ranked by a metric such as significance, true peaks will have more similar ranks between replicates than peaks representing noise \cite{li2011}.

The IDR algorithm currently only works for two replicates. Future improvements could include expanding the IDR implementation to handle more replicates and improve the accuracy of peak calling with more data.

\subsubsection{ChIP-nexus peak calling}

\begin{wrapfigure}[19]{r}{6.02cm}
\centering
\includegraphics[width=6cm]{figures/figure3_tfiib-nexus-tata.pdf}
\caption{Average TFIIB ChIP-nexus signal from wild-type cells grown at 37\textdegree C for 80 minutes, aligned to 572 TATA boxes with no mismatches to the sequence TATAWAWR as previously defined by \citep{rhee2012}. The signal around each TATA box is scaled from 0 to 1 before taking the mean in order to normalize differences in levels of TFIIB binding. Crosslinking signal on the plus and minus strands are plotted above and below the x-axis, respectively.}
\label{fig:tfiib_tata}
\end{wrapfigure}

A number of tools have been created specifically for peak-calling using data from high-resolution ChIP techniques such as ChIP-nexus and ChIP-exo \cite{wang2014, hansen2016}. When applied to our TFIIB ChIP-nexus data, these tools tended to split what appeared to be a single TFIIB binding event into multiple peaks. This may be because TFIIB has been observed to crosslink to DNA at multiple points (Figure \ref{fig:tfiib_tata}) \cite{rhee2012}, which suggests that while these tools may work well for factors that bind symmetrically with a single crosslinking point on either side, there is still room for improvement when it comes to factors with more complex binding patterns. For the purposes of this project, the standard ChIP-seq peak caller MACS2 was used \cite{zhang2008}.

ChIP-seq peaks lack strand information, as DNA binding factors usually do not bind DNA in a strand-specific manner. Because of this, we could not separate intragenic TFIIB peaks into peaks associated with sense or antisense transcription. The distinctive shape of the aggregate TFIIB ChIP-nexus signal (Figure \ref{fig:tfiib_tata}) suggests that information about the strand of transcription may be present in the ChIP-nexus binding profile. Future work could include learning the direction of transcription from labeled ChIP-nexus training data.

\subsection{TSS-seq and TFIIB ChIP-nexus results from \textit{spt6-1004}}

\begin{figure}[H]
\centering
\includegraphics[width=17.4cm]{figures/figure2_tss-seq-heatmaps.pdf}
\caption{Heatmaps of sense and antisense TSS-seq signal from wild-type and \textit{spt6-1004} cells, over 3522 non-overlapping genes aligned by wild-type genic TSS and sorted by annotated transcript length. Data are shown for each gene up to 300 nucleotides 3' of the cleavage and polyadenylation site (CPS), indicated by the white dotted line. Values are the mean of spike-in normalized coverage in non-overlapping 20 nucleotide bins, averaged over two replicates. Values above the 93rd percentile are set to the 93rd percentile for visualization.}
\label{fig:tss_heatmaps}
\end{figure}

\begin{wrapfigure}[23]{R}{8.72cm}
% \begin{SCfigure}[0.7][h]
\centering
\includegraphics[width=8.7cm]{figures/figure4_tfiib-heatmaps.pdf}
\caption{Heatmaps of TFIIB binding measured by ChIP-nexus, over the same regions shown in Figure \ref{fig:tss_heatmaps}. Values are the mean of library-size normalized coverage in non-overlappin 20 bp bins, averaged over two replicates. Values above the 85th percentile are set to the 85th percentile for visualization.}
\label{fig:tfiib_heatmaps}
% \end{SCfigure}
\end{wrapfigure}

To assay transcription start sites and transcription initiation in \textit{spt6-1004}, one of my collaborators performed TSS-seq and ChIP-nexus of TFIIB. In wild-type cells, TSS-seq and TFIIB ChIP-nexus signal has the expected distribution over the genome, with most TSS-seq signal at annotated genic TSSs and most TFIIB signal just upstream (Figures \ref{fig:tss_heatmaps}, \ref{fig:tfiib_heatmaps}). In \textit{spt6-1004}, the signal for both assays infiltrates gene bodies, consistent with a role for intragenic initiation in the intragenic transcription phenotype. Notably, sense strand TSS-seq signal in \textit{spt6-1004} tends to occur towards the 3' end of genes, while antisense strand TSS-seq signal tends to occur towards the 5' ends of genes.

The TSS-seq data were quantified by peak calling and differential expression analysis, and classified into genomic categories based on their position relative to coding genes (Figure \ref{fig:tss_diffexp_summary}). The results from this analysis support the pattern observed in the heatmap visualization (Figure \ref{fig:tss_heatmaps}), with most genic TSSs downregulated and almost 8000 TSSs upregulated intragenic or antisense to genes. The overall effect of this on expression levels is to equalize expression levels between the different classes of transcripts (Figure \ref{fig:tss_expression_levels}).

\vspace{1.75cm}

\begin{figure}[H]
    \centering
    \begin{minipage}[t]{8.5cm}
        \centering
        \includegraphics[width=8.5cm]{figures/figure5_tss-diffexp-summary.pdf}
        \caption{Bar plots of the number of TSS-seq peaks differentially expressed in \textit{spt6-1004} versus wild-type.}
        \label{fig:tss_diffexp_summary}
    \end{minipage}\hfill
    \begin{minipage}[t]{8.5cm}
        \centering
        \includegraphics[width=8.5cm]{figures/figure6_tss-expression-levels.pdf}
        \caption{Violin plots of expression level distributions for different genomic classes of TSS-seq peaks in wild-type and \textit{spt6-1004}. Normalized counts are the mean of spike-in size factor normalized counts from two replicates.}
        \label{fig:tss_expression_levels}
    \end{minipage}
\end{figure}

The changes to binding of TFIIB in \textit{spt6-1004} are substantial. In wild-type cells, TFIIB ChIP-nexus signal is localized in discrete peaks in the promoter region of genes, while in \textit{spt6-1004}, many loci have TFIIB signal spread over a much broader region (Figure \ref{fig:tfiib_spreading}). The difference in binding pattern makes peak calling ineffective for quantifying TFIIB signal in this case: ChIP-seq peak callers generally use different algorithms for calling `narrow' peaks (e.g. for sequence-specific transcription factors) and `broad' peaks (e.g. for histone modifications), meaning that a single algorithm is unable to accurately call peaks in both wild-type and \textit{spt6-1004}. Therefore, to see whether changes in transcript levels in \textit{spt6-1004} correspond to changes in transcription initiation, we compared the change in TSS-seq signal at TSS-seq peaks in \textit{spt6-1004} to the change in TFIIB ChIP-nexus signal in the window extending 200 bp upstream of the TSS-seq peak. Changes in TSS-seq signal in \textit{spt6-1004} are associated with a change in TFIIB signal of the same sign at over 81\% of TSSs of any genomic class, indicating that the increase in intragenic transcript levels and decrease in genic transcript levels observed in \textit{spt6-1004} are in large part explained by changes in transcription initiation.

% \begin{wrapfigure}[18]{r}{11.02cm}
\begin{SCfigure}[50][h]
\centering
\includegraphics[width=11cm]{figures/figure8_tfiib-spreading-ssa4.pdf}
\caption[foo bar]{
    \begin{description}[align=right, nosep, itemindent=0pt, leftmargin=4.2em, font=\normalfont]
        \item [top)] TFIIB ChIP-nexus protection in wild-type and \textit{spt6-1004} strains over 20 kb of chromosome II flanking the \textit{SSA4} gene.
        \item [bottom)] Expanded view of TFIIB protection over the \textit{SSA4} gene.
    \end{description}
}
\label{fig:tfiib_spreading}
% \end{wrapfigure}
\end{SCfigure}

\begin{figure}[h]
\centering
\includegraphics[width=17.4cm]{figures/figure7_tss-v-tfiib.pdf}
\caption{Scatterplots of fold-change in \textit{spt6-1004} over wild-type, comparing TSS-seq and TFIIB ChIP-nexus. Each dot represents a TSS-seq peak paired with the window extending 200 nucleotides upstream of the TSS-seq peak summit for quantification of TFIIB ChIP-nexus signal. Fold-changes are regularized fold-change estimates from DESeq2, with size factors determined from the \textit{S. pombe} spike-in (TSS-seq), or \textit{S. cerevisiae} counts (ChIP-nexus).}
\end{figure}

\subsection{studying chromatin structure in \textit{spt6-1004} with MNase-seq}

To study the role that altered chromatin structure plays in the phenotypes seen in \textit{spt6-1004}, one of my collaborators performed MNase-sequencing in wild-type and \textit{spt6-1004}. To analyze the data, I developed a pipeline for paired-end MNase-seq data (\href{https://github.com/winston-lab/mnase-seq}{github.com/winston-lab/mnase-seq}) that includes quantification and visualization of changes in nucleosome properties, among other MNase-seq specific steps.

\begin{figure}[H]
    \centering
    \begin{minipage}[t]{8.5cm}
        \centering
        \includegraphics[width=8.5cm]{figures/figure9_mnase-metagene.pdf}
        \caption{Average MNase-seq dyad signal in wild-type and \textit{spt6-1004}, over 3522 non-overlapping genes. Values are the mean of spike-in normalized coverage in non-overlapping 20 bp bins, averaged over two replicates (\textit{spt6-1004}) or one experiment (wild-type). The solid line and shading are the median and the inter-quartile range.}
        \label{fig:mnase_metagene}
    \end{minipage}\hfill
    \begin{minipage}[t]{8.5cm}
        \centering
        \includegraphics[width=8.5cm]{figures/figure10_global-nuc-fuzz-occ.pdf}
        \caption{Contour plot of the global distributions of nucleosome occupancy and fuzziness in wild-type and \textit{spt6-1004}. Dashed lines indicate median values.}
        \label{fig:global_nuc_fuzz}
    \end{minipage}
\end{figure}

In wild-type, the MNase-seq data recapitulates the expected patterns of a nucleosome depleted region upstream of a strongly positioned +1 nucleosome, and a regularly phased pattern of nucleosomes over gene bodies (Figure \ref{fig:mnase_metagene}). In \textit{spt6-1004}, nucleosome signal is severely reduced at canonical nucleosome positions and spreads into inter-nucleosome regions. A change in aggregate nucleosome signal such as that observed in Figure \ref{fig:mnase_metagene} can result from many combinations of changes to nucleosome occupancy (the number of reads assigned to a nucleosome), fuzziness (the standard deviation of read positions for a nucleosome), and position (the coordinate with the maximum reads for a nucleosome) \cite{chen2013}. Using DANPOS2 \cite{chen2013}, I called nucleosome positions and quantified these metrics for wild-type and \textit{spt6-1004}. Nucleosomes in wild-type span a relatively wide range of occupancy and fuzziness space, with highly occupied nucleosomes tending to be less fuzzy (i.e. more well-positioned) (Figure \ref{fig:global_nuc_fuzz}). In \textit{spt6-1004}, the population is much more homogeneous: nucleosome occupancy is decreased globally, and nucleosome fuzziness is restricted to the high end of the wild-type distribution.

Previous studies observed two trends: 1) In wild-type cells, nucleosome positioning is weaker over highly transcribed genes than over moderately transcribed genes \cite{shivaswamy2008}, and 2) In \textit{spt6-1004} cells, the decrease in nucleosome occupancy is greater for highly transcribed genes \cite{ivanovska2011}. To re-examine these trends, we looked at the MNase-seq data in the context of NET-seq data, which reports the position of actively transcribing RNAPII and reflects a gene's level of transcription (Figure \ref{fig:mnase_heatmap}) \cite{churchman2012}. The data support the first trend: in wild-type, genes with the strongest NET-seq signal have decreased MNase-seq signal. However, there is no obvious relationship between transcription level and the nucleosome changes observed in \textit{spt6-1004} (Figure \ref{fig:mnase_heatmap}). The apparent discrepancy might be explained by the improved resolution and breadth of MNase-seq versus the MNase and microarray of chromosome III used in the previous study \cite{ivanovska2011}.

\begin{figure}[H]
\centering
\includegraphics[width=17.4cm]{figures/figure11_mnase-heatmap.pdf}
\caption[foo bar]{
    \begin{description}[align=right, nosep, itemindent=0pt, leftmargin=4.2em, font=\normalfont]
        \item [left)] Heatmap of sense strand NET-seq signal for 3522 non-overlapping genes, aligned by genic TSS and sorted by total sense strand NET-seq signal in the window extending 500 nucleotides downstream from the genic TSS. Values are the mean of library-size normalized coverage in non-overlapping 20 nt bins, averaged over two replicates.
        \item [middle)] Heatmaps of MNase-seq dyad signal in wild-type and \textit{spt6-1004} for the same genes, aligned by wild-type +1 nucleosome dyad and arranged by sense NET-seq signal as in the leftmost panel. Values are the mean of spike-in normalized coverage in non-overlapping 20 bp bins, averaged over two replicates (\textit{spt6-1004}) or one experiment (wild-type).
        \item [right)] Heatmaps of fold-change in nucleosome occupancy and fuzziness for the same genes, aligned by wild-type +1 nucleosome dyad and arranged by sense NET-seq signal as in the leftmost panel.
    \end{description}
}
\label{fig:mnase_heatmap}
\end{figure}

\subsection{features of \textit{spt6-1004} intragenic promoters}

The resolution with which we were able to identify intragenic transcription start sites allowed us to closely examine the chromatin and sequence features of intragenic promoters, and to compare intragenic to genic promoters. We made this comparison for MNase-seq data, DNA sequence information content, and sequence motifs.

\subsubsection{clustering MNase-seq data at intragenic TSSs}

The average MNase-seq signal around all intragenic TSSs significantly upregulated in \textit{spt6-1004} showed no periodicity, suggesting the existence of multiple nucleosome patterns. To separate intragenic TSSs into groups with similar nucleosome patterning, I used the wild-type and \textit{spt6-1004} MNase-seq data flanking the intragenic TSSs to train a self-organizing map, an unsupervised learning method used to produce low-dimensional representations of an input space \cite{wehrens2007}. First, spike-in normalized MNase-seq dyad signal in the window $\pm$150 bp from the summits of the 6059 intragenic TSS-seq peaks was binned by taking the mean signal in non-overlapping 5 bp bins, and then averaged by taking the mean of two replicates (\textit{spt6-1004}) or one experiment (wild-type). The wild-type and \textit{spt6-1004} data were then used as equally weighted 6059$\times$60 input layers to a self-organizing map which assigned similar MNase-seq observations in 60-dimensional input space to similar nodes in a 2-dimensional (6$\times$8) rectangular grid. The 48 `code vectors' representing the typical MNase-seq pattern for each node were then clustered by agglomerative hierarchical clustering using sum of squares distance and Ward linkage. The resulting dendrogram was cut to produce two clusters of intragenic TSSs, whose aggregate MNase-seq signals differed primarily by phasing relative to the TSS (Figure \ref{fig:intra_mnase_metagene}).

\begin{figure}[H]
\centering
\includegraphics[width=17.4cm]{figures/figure12_intragenic-mnase.pdf}
\caption[foo bar]{
    \begin{description}[align=right, nosep, itemindent=0pt, leftmargin=6.2em, font=\normalfont]
        \item [top row)] Average MNase-seq dyad signal for two clusters of \textit{spt6-1004} intragenic TSSs (clustered by wild-type and \textit{spt6-1004} MNase-seq dyad signal flanking the TSS), as well as all genic TSSs detected in wild-type and \textit{spt6-1004}. Values are the mean of spike-in normalized dyad coverage in non-overlapping 10 bp bins, averaged over two replicates (\textit{spt6-1004}) or one experiment (wild-type). The solid line and shading are the median and inter-quartile range.
        \item [bottom row)] Average GC content of the DNA sequence, as above.
    \end{description}
}
\label{fig:intra_mnase_metagene}
\end{figure}

Similar to genic TSSs, the two clusters of intragenic TSSs tend to lie on the border between regions of relative nucleosome enrichment and depletion. However, they lack the relatively wide nucleosome-depleted region characteristic of genic promoters. Since low-GC sequences are known to be unfavorable for nucleosomes, we examined the GC content of the DNA sequence at intragenic TSSs. For cluster two, GC content drops just upstream of the intragenic TSS, in the same region where MNase-seq signal is depleted. This depletion is similar in position but smaller in magnitude as the GC depletion upstream of genic TSSs (Figure \ref{fig:intra_mnase_metagene}). Overall, nucleosome positioning is weaker at intragenic promoters compared to genic promoters, consistent with the tendency of intragenic promoters to occur towards the 3' end of genes, where positioning is known to decrease.

\pagebreak

\subsubsection{information content of intragenic TSSs}

\begin{wrapfigure}[12]{R}{8.72cm}
\centering
\includegraphics[width=8.7cm]{figures/figure13_seqlogos.pdf}
\caption{Sequence logos of the information content of TSS-seq reads overlapping genic and intragenic TSS-seq peaks in \textit{spt6-1004}.}
\label{fig:seqlogos}
\end{wrapfigure}

To examine differences in DNA sequence at intragenic TSSs compared to genic TSSs, we aligned the sequences for all TSS-seq reads overlapping TSS-seq peaks of each class, and calculated the information content of each type of TSS. Intragenic TSSs have a sequence motif that is almost identical to the sequence motif previously observed for genic TSSs (Figure \ref{fig:seqlogos}). This suggests that RNA polymerase initiates transcription similarly at genic and intragenic TSSs, and that the lack of intragenic initiation in wild-type is due to inaccessibility of the initiation motif, possibly due to being wrapped in histones.

\subsubsection{enrichment of motifs at intragenic TSSs}

\begin{wrapfigure}[13]{r}{8.62cm}
\centering
\includegraphics[width=8.6cm]{figures/figure14_intragenic-tata.pdf}
\caption{Scaled density of occurrences of exact matches to the motif TATAWAWR upstream of TSSs. For each category, a Gaussian kernel density estimate of the positions of motif occurrences is multiplied by the number of motif occurrences in the genomic category and divided by the number of regions in the category.}
\label{fig:tata}
\end{wrapfigure}

To examine whether sequence-specific transcription factors contribute to the expression of intragenic transcripts in \textit{spt6-1004}, we looked for enrichment or depletion of the DNA sequence motifs associated with these factors upstream of intragenic TSSs. Exact matches to the TATA element consensus sequence TATAWAWR are enriched upstream between 100 and 150 nt upstream of intragenic TSSs, in the same position but to a lesser degree than the TATA enrichment observed upstream of genic TSSs (Figure \ref{fig:tata}). This further supports the model that \textit{spt6-1004} intragenic promoters are sequences similar to canonical genic promoters, which become accessible for transcription initiation when the normal chromatin state is disturbed.

\vspace{6em}

\newpage
\section{functions of intragenic transcription in stress}

\subsection{collaborators}

\begin{description}[align=right, labelwidth=5cm, noitemsep]
    \item [Steve Doris] generated TSS-seq and ChIP-nexus libraries
    \item [Dan Spatt] polyribosome fractionation
\end{description}

\subsection{introduction to wild-type intragenic transcription}

In the \textit{spt6-1004} mutant, intragenic transcription is most likely spurious transcription that occurs when disruptions in chromatin make promoter-like sequences accessible to the transcription initiation machinery. In wild-type cells, intragenic transcription is much more limited, but there is evidence for specific instances of intragenic transcription to be biologically functional \cite{mcknight2014, gammie1999}. In one example, DNA replication stress induces intragenic transcription of the \textit{ASE1} mitotic spindle microtubule bundling gene \cite{mcknight2014}. Translation of the \textit{ASE1} intragenic transcript produces a truncated protein containing a C-terminal microtubule-binding domain but lacking an N-terminal dimerization domain. Because the short protein binds microtubules but does not dimerize, it antagonizes the function of the full length Ase1 protein, a function which is important for the cellular response to replication stress.

Even if intragenic transcription does not act through the production of truncated protein, the possibility for either the process of intragenic transcription or the intragenic RNA to be important to a cell is supported by numerous examples of the regulatory role of noncoding transcription and noncoding RNA. Conversely, it is entirely possible for intragenic transcription to have no function and simply exist as an evolutionarily neutral coincidence of DNA sequence. In this project, we are attempting to identify and characterize biologically functional cases of intragenic transcription in wild-type \textit{S. cerevisiae}. Experimentally determining whether a case of intragenic transcription has a function is costly because the search space is large: Negative results indicating that the transcription is non-functional can occur if transcription truly is spurious, but also if the conditions tested aren't the conditions in which the transcription is functional. This makes it especially important to limit experiments to the cases of intragenic transcription that are most likely to have a function. My collaborators have generated three datasets which we are using to identify the best experimental candidates.

\subsection{TFIIB ChIP-nexus in stress conditions}

A straightforward way to begin to link intragenic transcription to a phenotype is to perturb cells and find instances of intragenic transcription with expression changes following the perturbation. To do this, one of my collaborators has performed TFIIB ChIP-nexus for yeast cells in three stress conditions: oxidative stress (by addition of diamide to the media), amino acid starvation, and nitrogen starvation. In each condition, we identify upregulated intragenic TFIIB peaks (Figure \ref{fig:stress_tfiib-heatmaps}). Some peaks are upregulated in more than one condition; these may be peaks relevant to more general stress responses. Because we rely on the existing annotation of coding sequences to classify peaks as intragenic, there are undoubtedly peaks which are classified as intragenic due to annotation errors. However, the number of peaks is small enough that manually verifying individual cases is reasonable. As we have done for other datasets, we will be searching for DNA motifs proximal to intragenic peaks to find binding sites of sequence-specific transcription factors which may be responsible for regulating cases of intragenic transcription. Such binding sites will be useful targets for mutation when testing the function of intragenic transcription.

\begin{figure}[H]
\centering
\includegraphics[width=17.4cm]{figures/stress_figure1-tfiib-heatmaps.pdf}
\caption{Heatmaps of TFIIB ChIP-nexus protection in unstressed, oxidative stress, amino acid starvation, and nitrogen starvation conditions, over genes that contain an intragenic TFIIB peak induced in a stress condition. Genes are aligned by their genic TSS, and sorted by the distance between the genic TSS and the intragenic TFIIB peak. Values are the mean of library-size normalized coverage in non-overlappin 10 bp bins, averaged over two replicates. Values above the 90th percentile are set to the 90th percentile for visualization.}
\label{fig:stress_tfiib-heatmaps}
\end{figure}

\subsection{TSS-seq of different yeast species in oxidative stress}

It is reasonable to assume that intragenic transcription that is evolutionarily conserved is more likely to be functional. We cannot only consider DNA sequence conservation of intragenic promoters: intragenic promoters by definition lie within coding sequences, which will be highly conserved. To generate data on conservation of intragenic transcription, one of my collaborators performed TSS-seq for oxidative stress in \textit{S. cerevisiae} and the \textit{Saccharomyces sensu stricto} species \textit{S. mikatae} and \textit{S. bayanus var. uvarum}. To determine which instances of intragenic transcription are conserved between these species, it is necessary to map the \textit{cerevisiae} genome sequence to the other two genome sequences. We are doing this using the LASTZ pairwise genome alignment tool \cite{harris2007}, and are currently working on finding conserved cases of intragenic transcription in oxidative stress.

\subsection{polyribosome-associated TSS-seq in oxidative stress}

As previously noted, intragenic transcripts can potentially act through translation into truncated protein. To discover intragenic transcripts where this might be the case in \textit{S. cerevisiae} under oxidative stress, my collaborators performed sucrose gradient fractionation and sequenced TSS-seq libraries constructed from the RNA associated with the polyribosome fraction. These libraries represent transcription start sites which are associated with the translation machinery and are thus likely to be translated. Intragenic transcripts which are translated are attractive targets for experimentation because preventing expression of the intragenic protein is easily achieved by silently mutating the intragenic start codon. We are currently analyzing the data to identify these targets. Additionally, by normalizing results from polyribosome-associated TSS-seq to standard TSS-seq from total RNA, we may be able to extract information on what features make an intragenic transcript more or less likely to be translated.

\newpage
\section{genomics of transcription elongation factor Spt5}

\subsection{collaborators}

\begin{description}[align=right, labelwidth=5cm, noitemsep]
    \item [Ameet Shetty] generated TSS-seq, MNase-seq, NET-seq, RNA-seq, and ChIP-seq libraries
\end{description}

\subsection{introduction to Spt5 and techniques used to study it}

My collaborators on this project are interested in better understanding the transcription elongation factor \textbf{Spt5}. The following is a quick introduction to Spt5 \cite{shetty2017}:

\begin{itemize}[nosep, topsep=.5em]
    \item Spt5 is the only transcription factor known to be conserved in all three domains of life \cite{hartzog2013, werner2012}.
    \item Spt5 co-localizes with elongating RNAPII \cite{mayer2010, rahl2010}.
    \item Spt5 binds over the RNAPII clamp domain, likely stabilizing the elongation complex \cite{hirtreiter2010, klein2011, martinez-rucobo2011}.
    \item Spt5 physically recruits factors to the elongating transcription complex, in a manner dependent on the modification status of its C-terminal region (CTR) \cite{hartzog2013}:
    \begin{itemize}[nosep]
        \item in its unphosphorylated state, the CTR aids in recruiting the mRNA capping enzyme \cite{doamekpor2014, doamekpor2015, schneider2010, wen1999}
        \item in its phosphorylated state, the CTR recruits the Paf1 complex, which is important for RNAPII elongation \cite{liu2009, mbogning2013, wier2013, zhou2009}
        \item Spt5 helps to recruit mRNA 3' end processing factors \cite{mayer2012, stadelmayer2014, yamamoto2014}.
        \item Spt5 helps to recruit the Rpd3S histone deacetylase complex \cite{drouin2010}.
    \end{itemize}
\end{itemize}

\begin{wrapfigure}[16]{r}{8.52cm}
\centering
\includegraphics[width=8.5cm]{figures/spt5_figure1-netseq-metagene.pdf}
\caption{Average sense strand NET-seq signal in Spt5 non-depleted and depleted cells, over 1989 non-overlapping coding genes scaled to the same length. Values are the mean of spike-in normalized coverage over two replicates. The solid line and shading are the median and the inter-quartile range.}
\label{fig:spt5_netseq-metagene}
\end{wrapfigure}

The strategy my collaborator uses to study Spt5 is to apply genomic techniques to \textit{Schizosaccharomyces pombe} cells that are depleted of Spt5 protein. Spt5 is essential for viability in \textit{S. pombe}, so depletion of Spt5 cannot be achieved by using a deletion mutant. Instead, my collaborator uses a strain in which expression of Spt5 can be shut off at two levels: At the transcription level, Spt5 is under the control of a thiamine-repressible promoter, such that transcription of Spt5 can be shut off with the addition of thiamine to the media. At the protein level, the Spt5 protein is tagged with an auxin-inducible degron, such that Spt5 is degraded upon addition of auxin to the media. For most experiments, cells are collected 4.5 hours after the addition of thiamine and auxin to the media, a timepoint at which Spt5 is depleted to around 13\% of non-depleted levels but the cells are still viable \cite{shetty2017}.

Using this system, my collaborator applied a battery of genomic assays to cells in which Spt5 is depleted, including ChIP-seq of RNAPII, NET-seq, RNA-seq, and 4tU-seq. The results of these assays (not analyzed by me, although I am re-analyzing these datasets) are published \cite{shetty2017}, and can be summarised as follows: 1) Upon depletion of Spt5, RNAPII accumulates at the 5' end of genes (Figure \ref{fig:spt5_netseq-metagene}), consistent with the role of Spt5 as a factor required for transcription elongation. 2) Upon depletion of Spt5, many transcripts antisense to coding genes become upregulated, particularly towards the 5' end of genes (Figure \ref{fig:spt5_rnaseq-heatmaps}).

\begin{SCfigure}[50][h]
\centering
\includegraphics[width=11cm]{figures/spt5_figure2-rnaseq-heatmaps.pdf}
\caption{Heatmaps of antisense RNA-seq signal in Spt5 non-depleted and depleted cells, over 1989 non-overlapping coding genes aligned by sense strand genic TSS and sorted by transcript length. Data are shown for each gene up to 300 nt downstream of the cleavage and polyadenylation site (CPS), indicated by the dotted white line. Values are the mean of library-size normalized coverage in non-overlapping 20 nt bins, averaged over two replicates. Values above the 90th percentile are set to the 90th percentile for visualization.}
\label{fig:spt5_rnaseq-heatmaps}
\end{SCfigure}

% \begin{figure}[H]
%     \centering
%     \begin{minipage}[c]{8.5cm}
%         \centering
%         \includegraphics[width=8.5cm]{figures/spt5_figure1-netseq-metagene.pdf}
%         \caption{Average sense strand NET-seq signal in Spt5 non-depleted and depleted cells, over 1989 non-overlapping genes scaled to the same length. Values are the mean of spike-in normalized coverage over two replicates. The solid line and shading are the median and the inter-quartile range.}
%         \label{fig:spt5_netseq-metagene}
%     \end{minipage}\hfill
%     \begin{minipage}[c]{8.5cm}
%         \centering
%         \includegraphics[width=8.5cm]{figures/spt5_figure2-rnaseq-heatmaps.pdf}
%         \caption{Heatmaps of antisense RNA-seq signal in Spt5 non-depleted and depleted cells, over 1989 non-overlapping genes sorted by transcript length and aligned by annotated sense TSS. Values are the mean of library-size normalized coverage in non-overlapping 20 nt bins, averaged over two replicates. Values above the 90th percentile are set to the 90th percentile for visualization.}
%         \label{fig:spt5_rnaseq-heatmaps}
%     \end{minipage}
% \end{figure}

\subsection{TSS-seq and MNase-seq results in Spt5 depletion}

Following the publication, my collaborator performed two more assays to characterize the upregulation of Spt5-depletion antisense transcripts in more detail: TSS-seq to map the start sites of these transcripts, and MNase-seq to ask whether their upregulation is associated with changes in chromatin.

\begin{figure}[H]
\centering
\includegraphics[width=17.4cm]{figures/spt5_figure3-antisense-heatmaps.pdf}
\caption{Heatmaps of antisense TSS-seq, RNA-seq, and NET-seq signal in Spt5 non-depleted and depleted cells, over 1355 genes with a significantly upregulated antisense TSS-seq peak, aligned by sense TSS and arranged by the distance from the sense TSS to the upregulated antisense peak. Values are the mean of spike-in (TSS-seq and NET-seq) or library-size (RNA-seq) normalized coverage in non-overlapping 20 nt bins, averaged over two or more replicates. Values above the 0.995 (TSS-seq), 0.96 (RNA-seq), and 0.95 (NET-seq) quantiles are set to their respective quantiles for visualization.}
\label{fig:spt5_antisense-heatmaps}
\end{figure}

Using TSS-seq, we identify around 1300 TSSs that are significantly upregulated in Spt5 depletion and lie antisense to a coding transcript (Figure \ref{fig:spt5_antisense-heatmaps}). The majority of these antisense TSSs occur within 1 kb downstream of the corresponding sense TSS, and most are associated with elevated antisense RNA-seq and NET-seq signal in the depletion condition (Figure \ref{fig:spt5_antisense-heatmaps}). In order to do more quantitative comparisons between the assays, we will be performing transcript annotation using existing tools to annotate these transcripts by RNA-seq and NET-seq. We will also be using the TSS positions to search for DNA motifs which may be associated with these transcripts. There are no readily available databases of known motifs for \textit{S. pombe}, so we will be doing \textit{de novo} motif discovery using existing tools.

\begin{wrapfigure}[13]{r}{8.72cm}
\centering
\includegraphics[width=8.7cm]{figures/spt5_figure4-mnase-metagene.pdf}
\caption{Average MNase-seq dyad signal in Spt5 non-depleted and depleted cells, over 1985 non-overlapping coding genes aligned by +1 nucleosome dyad. Values are the mean of library-size normalized coverage in non-overlapping 5 bp bins, averaged over biological triplicates. The solid line and shading are the median and the inter-quartile range.}
\label{fig:spt5_mnase-metagene}
\end{wrapfigure}

The initial MNase-seq results in Spt5 depletion are summarized in Figure \ref{fig:spt5_mnase-metagene}. Nucleosome signal is reduced at canonical nucleosome positions and increased in inter-nucleosomal spaces. Notably, these effects seem to mainly occur downstream of the +1 nucleosome, though it should be noted that the lack of spike-in normalization in this experiment means that nucleosome signal could be globally increased or decreased in reality. We have separated the changes in MNase-seq signal into changes in nucleosome occupancy, position, and fuzziness as we have done for \textit{spt6-1004}, and will be analyzing these results in more detail.

\vspace{1.6cm}
\subsection{chromatin structure at Spt5-depletion antisense TSSs}

\begin{wrapfigure}[21]{r}{8.72cm}
\centering
\includegraphics[width=8.7cm]{figures/spt5_figure5-antisense-mnase-metagene.pdf}
\caption[foo bar]{
    \begin{description}[align=right, nosep, itemindent=0pt, leftmargin=4.2em, font=\normalfont]
        \item [top)] Average MNase-seq dyad signal in Spt5 non-depleted and depleted cells, over 1312 antisense TSSs upregulated in Spt5 depletion. Values are the mean of library-size normalized coverage in non-overlapping 10 bp bins, averaged over biological triplicates. The solid line and shading are the median and the inter-quartile range.
        \item [bottom)] Average GC content of the DNA, over the same regions.
    \end{description}
}
\label{fig:spt5_antisense-mnase-metagene}
\end{wrapfigure}

The average MNase-seq signature for all antisense Spt5-depletion TSSs is shown in Figure \ref{fig:spt5_antisense-mnase-metagene}. There is moderate periodicity in the signal even without clustering, indicating that the nucleosome signature around the antisense TSSs is relatively uniform. Spt5-depletion TSSs tend to occur between canonical nucleosome positions, likely in the internucleosomal spaces between the +1, +2, and +3 nucleosomes, based on the distance these TSSs occur from the sense strand genic TSS (Figure \ref{fig:spt5_antisense-heatmaps}). Interestingly, GC content is increased in the region of nucleosome depletion where the antisense TSS is (Figure \ref{fig:spt5_antisense-mnase-metagene}), in contrast to the expected decrease due to the nucleosome-disfavoring properties of low-GC sequences (compare to GC content at \textit{S. cerevisiae} TSSs in Figure \ref{fig:mnase_metagene}). We will be clustering the MNase-seq data similar to what we have done in \textit{spt6-1004} to refine the picture of the nucleosome landscape at antisense TSSs, following up on sequence properties with \textit{de novo} motif finding, and quantifying changes in nucleosome properties around the TSSs.

\newpage
% \printbibliography
\bibliography{prospectus}{}
\bibliographystyle{plain}
\end{document}

