\documentclass[9pt, letterpaper]{article}
\usepackage[letterpaper, margin=21mm]{geometry}

\usepackage{mathspec}
\setmainfont{Roboto-Regular}[
    Path=fonts/,
    BoldFont=Roboto-Bold,
    ItalicFont=Roboto-Italic,
    BoldItalicFont=Roboto-BoldItalic
]
% \setmathsfont(Digits)[Path=fonts/]{Roboto-Regular}
\setmathrm[Path=fonts/]{Roboto-Regular}

% \usepackage[style=nature]{biblatex}
\usepackage[]{natbib}
% \addbibresource{prospectus.bib}

\usepackage{morefloats}
\usepackage{enumitem}
\usepackage{graphicx}
\usepackage{float}
\usepackage{sidecap}
\usepackage{wrapfig}
\usepackage{hyperref}
\usepackage{textcomp}
\usepackage{caption}
\captionsetup{font=footnotesize, singlelinecheck=off}
\usepackage{lipsum}
% \usepackage{wrapfig}

\setlength\intextsep{4pt}

\begin{document}

\begin{titlepage}
\begin{tabular}{ r l }
 proposed title:   & Digital plumbing in the genomics era. \\
 		   & \\
 author:	   & James Chuang \\
 		   & \\
 research advisor: & Fred Winston \\
 		   & \\
 abstract:	   & Lorem ipsum.
\end{tabular}
\end{titlepage}

\tableofcontents
\newpage

\section{introduction}

Within the past two years, I have taken on the three projects described in this prospectus document. My role in these projects is best described as \textbf{data engineer}: I build pipelines for processing large (in this case, genomic) datasets, from raw data to statistical analysis and data visualization. For every step of a pipeline, this involves surveying available tools and selecting the tools that are most suitable. In some cases, existing tools are insufficient for a task, so I write code to complete the task. All of my data analyses are open source (\url{github.com/winston-lab}) and designed to be reproducible by others: For every publication, I upload an archive that allows anyone to go from the raw data to the figures in the publication (see \url{https://doi.org/10.5281/zenodo.1403167} for an example).

\section{genomics of transcription elongation factor Spt6}

N.B. The work from this section is in press. A preprint is available at \url{https://doi.org/10.1101/347575} \cite{doris2018}.

\subsection{collaborators}

\begin{description}[align=right, labelwidth=5cm, noitemsep]
    \item [Steve Doris] optimized TSS-seq and ChIP-nexus protocols
    \item [] generated TSS-seq and ChIP-nexus libraries
    \item [Olga Viktorovskaya] generated MNase-seq libraries
    \item [Magdalenda Murawska] generated NET-seq libraries
    \item [Dan Spatt] wetlab experiments for publication
\end{description}

\subsection{introduction to Spt6, intragenic transcription, and assays used to study the two}

The aim of the work described in this section is related to understanding how a eukaryotic cell specifies which sites in its genome are permitted to become sites of transcription initiation. From past genetic studies in yeast, it is known that some of the factors involved in controlling the specificity of transcription initiation are actually transcription \textit{elongation} factors, including histone chaperones and histone modification enzymes \cite{kaplan2003, cheung2008, hennig2013}. My collaborators on this project are interested in the role of the transcription elongation factor \textbf{Spt6} in this process. The following is a quick introduction to Spt6 \cite{doris2018}:

\begin{itemize}[nosep, topsep=.5em]
\item Spt6 interacts directly with:
	\begin{itemize}[nosep]
	\item RNA polymerase II (RNAPII) \cite{close2011, diebold2011, liu2011, sdano2017, sun2010, yoh2007}
	\item histones \cite{bortvin1996, mccullough2015}
	\item the essential factor Spn1 (IWS1) \cite{diebold2010b, li2018, mcdonald2010}
	\end{itemize}
\item Spt6 is believed to function primarily as an elongation factor based on:
	\begin{itemize}[nosep]
	\item association with elongating RNAPII \cite{andrulis2000, ivanovska2011, kaplan2000, mayer2010}
	\item ability to enhance elongation in vitro \cite{endoh2004} and in vivo \cite{ardehali2009}
	\end{itemize}
\item Spt6 has been shown to regulate initiation in some cases \cite{adkins2006, ivanovska2011}
\item Spt6 regulates:
	\begin{itemize}[nosep]
	\item chromatin structure \cite{bortvin1996, degennaro2013, ivanovska2011, jeronimo2015, kaplan2003, perales2013, vanbakel2013}
	\item histone modifications, including:
		\begin{itemize}[nosep]
		\item H3K36 methylation \cite{carrozza2005, chu2006, yoh2008, youdell2008}
		\item in some organisms, H3K4 and H3K27 methylation \cite{begum2012, chen2012, degennaro2013, wang2017, wang2013}
		\end{itemize}
	\end{itemize}
\item Spt6 is likely to be a histone chaperone required to reassemble nucleosomes in the wake of transcription \cite{duina2011}.
\end{itemize}

Previous studies in the yeasts \textit{S. cerevisiae} and \textit{S. pombe} have examined the requirement for Spt6 in normal transcription \cite{cheung2008, degennaro2013, kaplan2003, pathak2018, uwimana2017, vanbakel2013}. Many of these studies make use of the same temperature-sensitive \textit{S. cerevisiae} \textit{spt6} mutant used in this project, \textbf{\textit{spt6-1004}}, in which Spt6 protein is depleted at the non-permissive temperature of 37\textdegree C \cite{kaplan2003}. The most notable phenotype of the \textit{spt6-1004} mutant is the expression of \textbf{intragenic transcripts}, transcripts which appear to start within protein-coding genes, both in the same orientation and in the antisense orientation relative to the coding gene \cite{cheung2008, degennaro2013, kaplan2003, uwimana2017}.

Previous genome-wide measurements of transcript levels in \textit{spt6-1004} were made using tiled microarrays \cite{cheung2008} and RNA sequencing \cite{uwimana2017}. These methods assay steady-state RNA levels, making them unable to determine whether the intragenic transcripts observed in \textit{spt6-1004} result from: A) new intragenic transcription initiation in the mutant, B) reduced decay of intragenic transcripts which are rapidly turned over in wild-type, or C) processing of full-length protein-coding RNAs. Additionally, these methods are suboptimal for identifying where intragenic transcription occurs, since the signal for an intragenic transcript in the same orientation as the gene it overlaps is convoluted with the signal from the full-length `genic' transcript \cite{cheung2008, lickwar2009}.

To overcome these issues, one of my collaborators applied two genomic assays to study transcription in \textit{spt6-1004}: transcription start-site sequencing (TSS-seq), and ChIP-nexus of TFIIB, a component of the RNA polymerase II pre-initiation complex (PIC). The TSS-seq technique used sequences the 5' end of capped and polyadenylated RNAs \cite{arribere2013, malabat2015}, allowing separation of intragenic from genic RNA signals and identification of intragenic transcript starts with single-nucleotide resolution. The ChIP-nexus technique used is a high-resolution chromatin immunoprecipitation technique in which the ChIPed DNA is exonuclease digested up to the bases crosslinked with the factor of interest before sequencing \cite{he2015}. When applied to the PIC component TFIIB, ChIP-nexus provides a way to determine whether intragenic transcripts result from new intragenic transcription initiation.

\subsection{pipeline development for TSS-seq and ChIP-nexus}

In order to use TSS-seq and ChIP-nexus to answer questions about Spt6 and intragenic transcription, I developed analysis pipelines for TSS-seq and ChIP-nexus data. The pipelines are written using the Python-based Snakemake workflow specification language \cite{koster2012}, and perform steps including read cleaning \cite{martin2011}, various quality controls \cite{andrews2012}, read alignment \cite{kim2013, langmead2012}, data normalization, coverage track generation \cite{quinlan2010}, peak calling \cite{zhang2008}, differential expression/binding analyses \cite{love2014}, data visualization, motif enrichment analyses \cite{bailey2015}, and gene ontology analyses \cite{young2010}. The Snakemake framework allows data analyses using these pipelines to be reproducible and scalable from workstations up to computing clusters. Up-to-date versions of these pipelines with more details on their capabilities are available at \href{https://github.com/winston-lab}{github.com/winston-lab}. In the following subsections I will only describe the thought behind a few of the more novel pipeline steps before moving on to results relating to Spt6 and intragenic transcription.

\subsubsection{TSS-seq peak calling}

TSS-seq data from a single region of transcription initiation tends to occur as a cluster of signal distributed over a range of positions, rather than a single nucleotide (Figure \ref{fig:tss_coverage}) \cite{arribere2013, malabat2015}. It is reasonable to consider such a cluster of TSS-seq signal as one entity, because the signals within the cluster are highly correlated to one another across different conditions. Therefore, to identify TSSs from TSS-seq data and quantify them for downstream analyses such as differential expression, it is necessary to first perform peak-calling on the data.

\begin{wrapfigure}[14]{l}{8.02cm}
\centering
\includegraphics[width=8cm]{figures/figure1_tss-seq-coverage.pdf}
\caption{Wild-type sense strand TSS-seq signal at the TEF1 genic TSS. Normalized counts are the mean of spike-in normalized coverage from two replicates.}
\label{fig:tss_coverage}
\end{wrapfigure}

In its current state, the TSS-seq pipeline calls peaks using 1-D watershed segmentation, followed by filtering for reproducibility by the Irreproducible Discovery Rate (IDR) method \cite{li2011}. First, a smoothed version of the TSS-seq coverage is generated for each sample using a discretized Gaussian kernel. Next, an initial set of peaks is generated by: 1) assigning all nonzero signal in the original, unsmoothed coverage to the nearest local maximum of the smoothed coverage, and 2) taking the minimum and maximum genomic coordinates of the original coverage assigned to each local maximum as the peak boundaries. The peaks are then trimmed to the smallest genomic window that includes 95\% of the original coverage, and the probability of the peak being generated by noise is estimated by a Poisson model where $\lambda$, the expected coverage, is the maximum of the expected coverage over the chromosome and the expected coverage in a window upstream of the peak (as for the ChIP-seq peak caller MACS2 \cite{zhang2008}). The influence of local read density on $\lambda$ is intended to reduce false positive peaks within gene bodies, especially for highly expressed genes: Since there are more fragments of RNA present for highly expressed genes, more fragments within the gene body will make it into the final library, even if they are not true 5' ends. To generate the final set of peaks, the peaks are ranked by significance under the Poisson model, and filtered by IDR. In brief, IDR attempts to separate true peaks from experimental noise based on the intuition that, when peaks in each replicate are independently ranked by a metric such as significance, true peaks will have more similar ranks between replicates than peaks representing noise \cite{li2011}.

The IDR algorithm currently only works for two replicates. Future improvements could include expanding the IDR implementation to handle more replicates and improve the accuracy of peak calling with more data.

\subsubsection{ChIP-nexus peak calling}

\begin{wrapfigure}[19]{r}{6.02cm}
\centering
\includegraphics[width=6cm]{figures/figure3_tfiib-nexus-tata.pdf}
\caption{Average TFIIB ChIP-nexus signal from wild-type cells grown at 37\textdegree C for 80 minutes, aligned to 572 TATA boxes with no mismatches to the sequence TATAWAWR as previously defined by \citep{rhee2012}. The signal around each TATA box is scaled from 0 to 1 before taking the mean in order to normalize differences in levels of TFIIB binding. Crosslinking signal on the plus and minus strands are plotted above and below the x-axis, respectively.}
\label{fig:tfiib_tata}
\end{wrapfigure}

A number of tools have been created specifically for peak calling using high-resolution ChIP techniques such as ChIP-nexus and ChIP-exo \cite{wang2014, hansen2016}. When applied to our TFIIB ChIP-nexus data, these tools tended to split what appeared to be a single TFIIB binding event into multiple peaks. This may be due to the fact that TFIIB has been observed to have multiple crosslinking points to the DNA \ref{fig:tfiib_tata} \cite{rhee2012}, and suggests that while these tools may work well for factors which bind symmetrically with a single crosslinking point on either side, there is still room for improvement when it comes to factors with more complex binding patterns. For the purposes of this project, the standard ChIP-seq peak caller MACS2 was used \cite{zhang2008}.

ChIP-seq peaks lack strand information, as DNA binding factors usually do not bind DNA in a strand-specific manner. Because of this, we could not separate intragenic TFIIB peaks into peaks associated with sense or antisense transcription. The distinctive shape of the aggregate TFIIB ChIP-nexus signal suggests that information about the strand of transcription may be present in the ChIP-nexus binding profile. Future work could include learning the direction of transcription from labeled ChIP-nexus training data.

\subsection{TSS-seq and TFIIB ChIP-nexus results in \textit{spt6-1004}}

\begin{figure}[H]
\centering
\includegraphics[width=17.4cm]{figures/figure2_tss-seq-heatmaps.pdf}
\caption{Heatmaps of sense and antisense TSS-seq signal from wild-type and \textit{spt6-1004} cells, over 3522 non-overlapping genes aligned by wild-type genic TSS and sorted by annotated transcript length. Data are shown for each gene up to 300 nucleotides 3' of the cleavage and polyadenylation site (CPS, indicated by the white dotted line). Values are the mean of spike-in normalized coverage in non-overlapping 20 nucleotide bins, averaged over two replicates. Values above the 93rd percentile are set to the 93rd percentile for visualization.}
\label{fig:tss_heatmaps}
\end{figure}

\begin{wrapfigure}[23]{R}{8.72cm}
% \begin{SCfigure}[0.7][h]
\centering
\includegraphics[width=8.7cm]{figures/figure4_tfiib-heatmaps.pdf}
\caption{Heatmaps of TFIIB binding measured by ChIP-nexus, over the same regions shown in Figure \ref{fig:tss_heatmaps}. Values are the mean of library-size normalized coverage in 20 basepair windows, averaged over two replicates. Values above the 85th percentile are set to the 85th percentile for visualization.}
\label{fig:tfiib_heatmaps}
% \end{SCfigure}
\end{wrapfigure}

To assay transcription start sites and transcription initiation in \textit{spt6-1004}, one of my collaborators performed TSS-seq and ChIP-nexus of TFIIB. In wild-type cells, TSS-seq and TFIIB ChIP-nexus signal has the expected distribution over the genome, with most TSS-seq signal at annotated genic TSSs and most TFIIB signal just upstream (Figures \ref{fig:tss_heatmaps}, \ref{fig:tfiib_heatmaps}). In \textit{spt6-1004}, the signal for both assays infiltrates gene bodies, consistent with a role for intragenic initiation in the intragenic transcription phenotype. Notably, sense strand TSS-seq signal in \textit{spt6-1004} tends to occur towards the 3' end of genes, while antisense strand TSS-seq signal tends to occur towards the 5' ends of genes.

The TSS-seq data were quantified by peak calling and differential expression analysis, and classified into genomic categories based on their position relative to coding genes (Figure \ref{fig:tss_diffexp_summary}). The results from this analysis support the pattern observed in the heatmap visualization (Figure \ref{fig:tss_heatmaps}), with most genic TSSs downregulated and almost 8000 TSSs upregulated intragenic or antisense to genes. The overall effect of this on expression levels is to equalize expression levels between the different classes of transcripts (Figure \ref{fig:tss_expression_levels}).

\vspace{1.75cm}

\begin{figure}[H]
    \centering
    \begin{minipage}[t]{8.5cm}
        \centering
        \includegraphics[width=8.5cm]{figures/figure5_tss-diffexp-summary.pdf}
        \caption[foo bar]{
            \begin{description}[align=right, nosep, itemindent=0pt, leftmargin=4.2em, font=\normalfont]
                \item [top)] Diagram of different genomic classes of TSSs.
                \item [bottom)] Bar plot showing the number of TSS-seq peaks differentially expressed in \textit{spt6-1004} versus wild-type.
            \end{description}
        }
        \label{fig:tss_diffexp_summary}
    \end{minipage}\hfill
    \begin{minipage}[t]{8.5cm}
        \centering
        \includegraphics[width=8.5cm]{figures/figure6_tss-expression-levels.pdf}
        \caption{Violin plots of expression level distributions for different genomic classes of TSS-seq peaks in wild-type and \textit{spt6-1004}. Normalized counts are the mean of \textit{S. pombe} spike-in size factor normalized counts from two replicates.}
        \label{fig:tss_expression_levels}
    \end{minipage}
\end{figure}

The changes to binding of TFIIB in \textit{spt6-1004} are substantial. In wild-type cells, TFIIB ChIP-nexus signal is localized in discrete peaks in the promoter region of genes, while in \textit{spt6-1004}, many loci have TFIIB signal spread over a much broader region (Figure \ref{fig:tfiib_spreading}). The difference in binding pattern makes peak calling ineffective for quantifying TFIIB signal in this case: ChIP-seq peak callers generally use different algorithms for calling `narrow' peaks (e.g. for sequence-specific transcription factors) and `broad' peaks (e.g. for histone modifications), meaning that a single algorithm is unable to accurately call peaks in both wild-type and \textit{spt6-1004}. Therefore, to see whether changes in transcript levels in \textit{spt6-1004} correspond to changes in transcription initiation, I compared the change in TSS-seq signal at TSS-seq peaks in \textit{spt6-1004} to the change in TFIIB ChIP-nexus signal in the window extending 200 bp upstream of the TSS-seq peak. Changes in TSS-seq signal in \textit{spt6-1004} are associated with a change in TFIIB signal of the same sign at over 81\% of TSSs of any genomic class, indicating that the increase in intragenic transcript levels and decrease in genic transcript levels observed in \textit{spt6-1004} are in large part explained by changes in transcription initiation.

% \begin{wrapfigure}[18]{r}{11.02cm}
\begin{SCfigure}[50][h]
\centering
\includegraphics[width=11cm]{figures/figure8_tfiib-spreading-ssa4.pdf}
\caption[foo bar]{
    \begin{description}[align=right, nosep, itemindent=0pt, leftmargin=4.2em, font=\normalfont]
        \item [top)] TFIIB ChIP-nexus protection in wild-type and \textit{spt6-1004} strains over 20 kb of chromosome II flanking the \textit{SSA4} gene.
        \item [bottom)] Expanded view of TFIIB protection over the \textit{SSA4} gene.
    \end{description}
}
\label{fig:tfiib_spreading}
% \end{wrapfigure}
\end{SCfigure}

\begin{figure}[h]
\centering
\includegraphics[width=17.4cm]{figures/figure7_tss-v-tfiib.pdf}
\caption{Scatterplots of fold-change in \textit{spt6-1004} over wild-type, comparing TSS-seq and TFIIB ChIP-nexus. Each dot represents a TSS-seq peak paired with the window extending 200 nucleotides upstream of the TSS-seq peak summit for quantification of TFIIB ChIP-nexus signal. Fold-changes are regularized fold-change estimates from DESeq2, with size factors determined from the \textit{S. pombe} spike-in (TSS-seq), or \textit{S. cerevisiae} counts (ChIP-nexus).}
\end{figure}

\subsection{studying chromatin structure in \textit{spt6-1004} with MNase-seq}

To study the role that altered chromatin structure plays in the phenotypes seen in \textit{spt6-1004}, one of my collaborators performed MNase-sequencing in wild-type and \textit{spt6-1004}. To analyze the data, I developed a pipeline for paired-end MNase-seq data (\href{https://github.com/winston-lab/mnase-seq}{github.com/winston-lab/mnase-seq}) that includes quantification and visualization of changes in nucleosome properties, among other MNase-seq specific steps.

In wild-type, the MNase-seq data recapitulates the expected patterns of a nucleosome depleted region upstream of a strongly positioned +1 nucleosome, and a regularly phased pattern of nucleosomes over gene bodies (Figure \ref{fig:mnase_metagene}). In \textit{spt6-1004}, nucleosome signal is severely reduced at canonical nucleosome positions and spreads into inter-nucleosome regions. A change in aggregate nucleosome signal such as that observed in Figure \ref{fig:mnase_metagene} can result from many combinations of changes to nucleosome occupancy (the number of reads assigned to a nucleosome), fuzziness (the standard deviation of read positions for a nucleosome), and position (the coordinate with the maximum reads for a nucleosome) \cite{chen2013}. Using DANPOS2 \cite{chen2013}, I called nucleosome positions and quantified these metrics for wild-type and \textit{spt6-1004}. Nucleosomes in wild-type span a relatively wide range of occupancy and fuzziness space, with highly occupied nucleosomes tending to be less fuzzy (i.e. more well-positioned) (Figure \ref{fig:global_nuc_fuzz}). In \textit{spt6-1004}, the population is much more homogeneous: nucleosome occupancy is decreased globally, and nucleosome fuzziness is restricted to the high end of the wild-type distribution.

\begin{figure}[h]
    \centering
    \begin{minipage}[t]{8.5cm}
        \centering
        \includegraphics[width=8.5cm]{figures/figure9_mnase-metagene.pdf}
        \caption{Average MNase-seq dyad signal in wild-type and \textit{spt6-1004}, over 3522 non-overlapping genes. Values are the mean of spike-in normalized coverage in nonoverlapping 20 basepair bins, averaged over two replicates (\textit{spt6-1004}) or one experiment (wild-type). The solid line and shading are the median and the inter-quartile range.}
        \label{fig:mnase_metagene}
    \end{minipage}\hfill
    \begin{minipage}[t]{8.5cm}
        \centering
        \includegraphics[width=8.5cm]{figures/figure10_global-nuc-fuzz-occ.pdf}
        \caption{Contour plot of the global distribution of nucleosome occupancy and fuzziness in wild-type and \textit{spt6-1004}. Dashed lines indicate median values.}
        \label{fig:global_nuc_fuzz}
    \end{minipage}
\end{figure}

Previous studies observed two trends: 1) In wild-type cells, nucleosome positioning is weaker over highly transcribed genes than over moderately transcribed genes \cite{shivaswamy2008}, and 2) In \textit{spt6-1004} cells, the decrease in nucleosome occupancy is greater for highly transcribed genes \cite{ivanovska2011}. To re-examine these trends, we looked at the MNase-seq data in the context of NET-seq data, which reports the position of actively transcribing RNAPII and reflects a gene's level of transcription (Figure \ref{fig:mnase_heatmap}) \cite{churchman2012}. The data support the first trend: in wild-type, genes with the strongest NET-seq signal have decreased MNase-seq signal. However, there is no obvious relationship between transcription level and the nucleosome changes observed in \textit{spt6-1004} (Figure \ref{fig:mnase_heatmap}). The apparent discrepancy might be explained by the improved resolution and breadth of MNase-seq versus the MNase and microarray of chromosome III used in the previous study \cite{ivanovska2011}.

\begin{figure}[H]
\centering
\includegraphics[width=17.4cm]{figures/figure11_mnase-heatmap.pdf}
\caption[foo bar]{
    \begin{description}[align=right, nosep, itemindent=0pt, leftmargin=4.2em, font=\normalfont]
        \item [left)] Heatmap of sense strand NET-seq signal for 3522 non-overlapping genes, aligned by genic TSS and sorted by total sense strand NET-seq signal in the window extending 500 nucleotides downstream from the genic TSS. Values are the mean of library-size normalized coverage in non-overlapping 20 nt bins, averaged over two replicates.
        \item [middle)] Heatmaps of MNase-seq dyad signal in wild-type and \textit{spt6-1004} for the same genes, aligned by wild-type +1 nucleosome dyad and arranged by sense NET-seq signal as in the leftmost panel. Values are the mean of spike-in normalized coverage in non-overlapping 20 bp bins, averaged over two replicates (\textit{spt6-1004}) or one experiment (wild-type).
        \item [right)] Heatmaps of fold-change in nucleosome occupancy and fuzziness for the same genes, aligned by wild-type +1 nucleosome dyad and arranged by sense NET-seq signal as in the leftmost panel.
    \end{description}
}
\label{fig:mnase_heatmap}
\end{figure}

\subsection{features of intragenic promoters}

The resolution with which we were able to identify intragenic transcription start sites allowed us to closely examine the chromatin and sequence features of intragenic promoters, and to compare intragenic to genic promoters. We made this comparison for MNase-seq data, DNA sequence information content, and sequence motifs.

\subsubsection{clustering MNase-seq data at intragenic TSSs}

The average MNase-seq signal around all intragenic TSSs significantly upregulated in \textit{spt6-1004} showed no periodicity, suggesting the existence of multiple nucleosome patterns. To separate intragenic TSSs into groups with similar nucleosome patterning, I used the wild-type and \textit{spt6-1004} MNase-seq data flanking the intragenic TSSs to train a self-organizing map, an unsupervised learning method used to produce low-dimensional representations of an input space \cite{wehrens2007}. First, spike-in normalized MNase-seq dyad signal in the window $\pm 150$ bp from the summits of the 6059 intragenic TSS-seq peaks was binned by taking the mean signal in non-overlapping 5 bp bins, and then averaged by taking the mean of two replicates (\textit{spt6-1004}) or one experiment (wild-type). The wild-type and \textit{spt6-1004} data were then used as equally weighted 6059$\times$ 60 input layers to a self-organizing map which assigning similar MNase-seq observations in 60-dimensional input space to similar nodes in a 2-dimensional (6 $\times$ 8) rectangular grid. The 48 `code vectors' representing the typical MNase-seq pattern for each node were then clustered by agglomerative hierarchical clustering using sum of squares distance and Ward linkage. The resulting dendrogram was cut to produce two clusters of intragenic TSSs, whose aggregate MNase-seq signals differed primarily by phasing relative to the TSS (Figure \ref{fig:intra_mnase_metagene}).

\begin{figure}[H]
\centering
\includegraphics[width=17.4cm]{figures/figure12_intragenic-mnase.pdf}
\caption[foo bar]{
    \begin{description}[align=right, nosep, itemindent=0pt, leftmargin=6.2em, font=\normalfont]
        \item [top row)] Average MNase-seq dyad signal for two clusters of \textit{spt6-1004} intragenic TSSs (clustered by the wild-type and \textit{spt6-1004} MNase-seq dyad signal flanking the TSS), as well as all genic TSSs detected in wild-type and \textit{spt6-1004}. Values are the mean of spike-in normalized dyad coverage in non-overlapping 10 bp bins, averaged over two replicates (\textit{spt6-1004}) or one experiment (wild-type). The solid line and shading are the median and inter-quartile range.
        \item [bottom row)] Average GC content of the DNA sequence, as above.
    \end{description}
}
\label{fig:intra_mnase_metagene}
\end{figure}

Similar to genic TSSs, the two clusters of intragenic TSSs tend to lie on the border between regions of relative nucleosome enrichment and depletion. However, they lack the relatively wide nucleosome-depleted region characteristic of genic promoters. Since AT-rich sequences are known to be unfavorable for nucleosomes, we examined the GC content of the DNA sequence at intragenic TSSs. For cluster two, GC content drops just upstream of the intragenic TSS, in the same region where MNase-seq signal is depleted. This depletion is similar in position but smaller in magnitude as the GC depletion upstream of genic TSSs (Figure \ref{fig:intra_mnase_metagene}). Overall, nucleosome positioning is weaker at intragenic promoters compared to genic promoters, consistent with the tendency of intragenic promoters to occur towards the 3' end of genes, where positioning is known to decrease.

\pagebreak

\subsubsection{information content of intragenic TSSs}

\begin{wrapfigure}[12]{R}{8.72cm}
\centering
\includegraphics[width=8.7cm]{figures/figure13_seqlogos.pdf}
\caption{Sequence logos of the information content of TSS-seq reads overlapping genic and intragenic TSS-seq peaks in \textit{spt6-1004}.}
\label{fig:seqlogos}
\end{wrapfigure}

To examine differences in DNA sequence at intragenic TSSs compared to genic TSSs, I aligned the sequences for all TSS-seq reads overlapping TSS-seq peaks of each class, and calculated the information content of each type of TSS. Intragenic TSSs have a sequence motif that is almost identical to the sequence motif previously observed for genic TSSs (Figure \ref{fig:seqlogos}). This suggests that RNA polymerase initiates transcription similarly at genic and intragenic TSSs, and that the lack of intragenic initiation in wild-type is due to inaccessibility of the initiation motif, possibly due to being wrapped in histones.

\subsubsection{enrichment of motifs at intragenic TSSs}

\begin{wrapfigure}[10]{r}{8.72cm}
\centering
\includegraphics[width=8.7cm]{figures/figure14_intragenic-tata.pdf}
\caption{Scaled density of occurrences of exact matches to the motif TATAWAWR upstream of TSSs. For each category, a Gaussian kernel density estimate of the positions of motif occurrences is multiplied by the number of motif occurrences in the genomic category and divided by the number of regions in the category.}
\label{fig:tata}
\end{wrapfigure}

To examine whether sequence-specific transcription factors contribute to the expression of intragenic transcripts in \textit{spt6-1004}, we looked for enrichment or depletion of the DNA sequence motifs associated with these factors upstream of intragenic TSSs. Exact matches to the TATA element consensus sequence TATAWAWR are enriched upstream between 100 and 150 nt upstream of intragenic TSSs, in the same position but to a lesser degree to the TATA enrichment observed upstream of genic TSSs (Figure \ref{fig:tata}).

\vspace{6em}

\section{functions of intragenic transcription in stress}

\subsection{collaborators}

\begin{description}[align=right, labelwidth=5cm, noitemsep]
    \item [Steve Doris] generated TSS-seq and ChIP-nexus libraries
    \item [Blake Tye] generated NET-seq and RNA-seq libraries
\end{description}

\subsection{introduction}

The aim of the project is to discover and characterize instances of intragenic transcription which are important in stress responses.

In the \textit{spt6-1004} mutant, intragenic transcription is most likely spurious transcription that occurs when disruptions in chromatin make promoter-like sequences accessible to the transcription initiation machinery. However, in some cases intragenic transcription may also be important in wild-type cells. Examples.

To discover intragenic transcription that may b



\section{genomics of transcription elongation factor Spt5}

\subsection{collaborators}

\begin{description}[align=right, labelwidth=5cm, noitemsep]
    \item [Ameet Shetty] generated TSS-seq, MNase-seq, NET-seq, RNA-seq, and ChIP-seq libraries
\end{description}

\subsection{introduction to Spt5 and techniques used to study it}

My collaborators on this project are interested in better understanding the transcription elongation factor Spt5. The following is a quick introduction to Spt5 \cite{shetty2017}:

\begin{itemize}[nosep, topsep=.5em]
    \item Spt5 is the only transcription factor known to be conserved in all three domains of life \cite{hartzog2013, werner2012}
    \item Spt5 co-localizes with elongating RNAPII \cite{mayer2010, rahl2010}
    \item Spt5 binds over the RNAPII clamp domain, likely stabilizing the elongation complex \cite{hirtreiter2010, klein2011, martinez-rucobo2011}
    \item Spt5 physically recruits factors to the elongating transcription complex, in a manner dependent on the modification status of its C-terminal region (CTR) \cite{hartzog2013}
    \begin{itemize}[nosep]
        \item in its unphosphorylated state, the CTR aids in recruiting the mRNA capping enzyme \cite{doamekpor2014, doamekpor2015, schneider2010, wen1999}
        \item in its phosphorylated state, the CTR recruits the Paf1 complex, which is important for RNAPII elongation \cite{liu2009, mbogning2013, wier2013, zhou2009}
        \item Spt5 helps to recruit mRNA 3' end processing factors \cite{mayer2012, stadelmayer2014, yamamoto2014}
        \item Spt5 helps to recruit the Rpd3S histone deacetylase complex \cite{drouin2010}
    \end{itemize}
\end{itemize}

The strategy my collaborator uses to study Spt5 is to apply genomic techniques to \textit{Schizosaccharomyces pombe} cells that are depleted of Spt5 protein. Spt5 is essential for viability in \textit{S. pombe}, so depletion of Spt5 cannot be achieved by using a deletion mutant. Instead, my collaborator uses a strain in which expression of Spt5 can be shut off at two levels: At the transcription level, Spt5 is under the control of a thiamine-repressible promoter, such that transcription of Spt5 can be shut off with the addition of thiamine to the media. At the protein level, the Spt5 protein is tagged with an auxin-inducible degron, such that Spt5 is degraded upon addition of auxin to the media. For most experiments, cells are collected 4.5 hours after the addition of thiamine and auxin to the media, a timepoint at which Spt5 is depleted to around 13\% of non-depleted levels but the cells are still viable \cite{shetty2017}.

% \begin{wrapfigure}[12]{r}{8.72cm}
% \centering
% \includegraphics[width=8.7cm]{figures/spt5_figure1-netseq-metagene.pdf}
% \caption{asdf}
% \label{fig:spt5_netseq-metagene}
% \end{wrapfigure}

Using this system, my collaborator applied a battery of genomic assays to cells in which Spt5 is depleted or non-depleted, including ChIP-seq of RNAPII, NET-seq, RNA-seq, and 4tU-seq. The results of these assays (not analyzed by me) are published \cite{shetty2017}, and can be summarised as follows: 1) Upon depletion of Spt5, RNAPII accumulates at the 5' end of genes (\ref{fig:spt5_netseq-metagene}), consistent with the role of Spt5 as a factor required for transcription elongation. 2) Upon depletion of Spt5, many transcripts antisense to coding genes become upregulated, particularly towards the 5' end of genes (\ref{fig:spt5_rnaseq-heatmaps}).

\begin{figure}[H]
    \centering
    \begin{minipage}[c]{8.5cm}
        \centering
        \includegraphics[width=8.5cm]{figures/spt5_figure1-netseq-metagene.pdf}
        \caption{Average sense strand NET-seq signal in Spt5 non-depleted and depleted cells, over 1989 non-overlapping genes scaled to the same length. Values are the mean of spike-in normalized coverage over two replicates. The solid line and shading are the median and the inter-quartile range.}
        \label{fig:spt5_netseq-metagene}
    \end{minipage}\hfill
    \begin{minipage}[c]{8.5cm}
        \centering
        \includegraphics[width=8.5cm]{figures/spt5_figure2-rnaseq-heatmaps.pdf}
        \caption{Heatmaps of antisense RNA-seq signal in Spt5 non-depleted and depleted cells, over 1989 non-overlapping genes sorted by transcript length and aligned by annotated sense TSS. Values are the mean of library-size normalized coverage in non-overlapping 20 nt bins, averaged over two replicates. Values above the 90th percentile are set to the 90th percentile for visualization.}
        \label{fig:spt5_rnaseq-heatmaps}
    \end{minipage}
\end{figure}

% \begin{wrapfigure}[23]{r}{8.72cm}
% \centering
% \includegraphics[width=8.7cm]{figures/spt5_figure2-rnaseq-heatmaps.pdf}
% \caption{asdf}
% \label{fig:spt5_rnaseq-heatmaps}
% \end{wrapfigure}

Following the publication, my collaborator wanted to characterize the upregulation of these antisense transcripts in more detail, and carried out two more assays: TSS-seq to precisely map the start sites of these transcripts, and MNase-seq to ask whether their upregulation is associated with changes in chromatin.

\begin{figure}[H]
\centering
\includegraphics[width=17.4cm]{figures/spt5_figure3-antisense-heatmaps.pdf}
\caption{asdf}
\label{fig:spt5_antisense-heatmaps}
\end{figure}

\begin{wrapfigure}[23]{l}{8.72cm}
\centering
\includegraphics[width=8.7cm]{figures/spt5_figure4-mnase-metagene.pdf}
\caption{asdf}
\label{fig:spt5_mnase-metagene}
\end{wrapfigure}

\begin{wrapfigure}[23]{r}{8.72cm}
\centering
\includegraphics[width=8.7cm]{figures/spt5_figure5-antisense-mnase-metagene.pdf}
\caption{asdf}
\label{fig:spt5_antisense-mnase-metagene}
\end{wrapfigure}

\newpage
% \printbibliography
\bibliography{prospectus}{}
\bibliographystyle{plain}
\end{document}

